<!DOCTYPE html>
<!-- START: inst/pkgdown/templates/layout.html --><!-- Generated by pkgdown: do not edit by hand --><html lang="en" data-bs-theme="auto">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<title>CRC1550 data management training material: All in One View</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<script src="../assets/themetoggle.js"></script><link rel="stylesheet" type="text/css" href="../assets/styles.css">
<script src="../assets/scripts.js" type="text/javascript"></script><!-- mathjax --><script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      config: ["MMLorHTML.js"],
      jax: ["input/TeX","input/MathML","output/HTML-CSS","output/NativeMML", "output/PreviewHTML"],
      extensions: ["tex2jax.js","mml2jax.js","MathMenu.js","MathZoom.js", "fast-preview.js", "AssistiveMML.js", "a11y/accessibility-menu.js"],
      TeX: {
        extensions: ["AMSmath.js","AMSsymbols.js","noErrors.js","noUndefined.js"]
      },
      tex2jax: {
        inlineMath: [['\\(', '\\)']],
        displayMath: [ ['$$','$$'], ['\\[', '\\]'] ],
        processEscapes: true
      }
    });
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><!-- Responsive Favicon for The Carpentries --><link rel="apple-touch-icon" sizes="180x180" href="../apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="manifest" href="../site.webmanifest">
<link rel="mask-icon" href="../safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#da532c">
<meta name="theme-color" media="(prefers-color-scheme: light)" content="white">
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="black">
</head>
<body>
    <header id="top" class="navbar navbar-expand-md top-nav incubator"><svg xmlns="http://www.w3.org/2000/svg" class="d-none"><symbol id="check2" viewbox="0 0 16 16"><path d="M13.854 3.646a.5.5 0 0 1 0 .708l-7 7a.5.5 0 0 1-.708 0l-3.5-3.5a.5.5 0 1 1 .708-.708L6.5 10.293l6.646-6.647a.5.5 0 0 1 .708 0z"></path></symbol><symbol id="circle-half" viewbox="0 0 16 16"><path d="M8 15A7 7 0 1 0 8 1v14zm0 1A8 8 0 1 1 8 0a8 8 0 0 1 0 16z"></path></symbol><symbol id="moon-stars-fill" viewbox="0 0 16 16"><path d="M6 .278a.768.768 0 0 1 .08.858 7.208 7.208 0 0 0-.878 3.46c0 4.021 3.278 7.277 7.318 7.277.527 0 1.04-.055 1.533-.16a.787.787 0 0 1 .81.316.733.733 0 0 1-.031.893A8.349 8.349 0 0 1 8.344 16C3.734 16 0 12.286 0 7.71 0 4.266 2.114 1.312 5.124.06A.752.752 0 0 1 6 .278z"></path><path d="M10.794 3.148a.217.217 0 0 1 .412 0l.387 1.162c.173.518.579.924 1.097 1.097l1.162.387a.217.217 0 0 1 0 .412l-1.162.387a1.734 1.734 0 0 0-1.097 1.097l-.387 1.162a.217.217 0 0 1-.412 0l-.387-1.162A1.734 1.734 0 0 0 9.31 6.593l-1.162-.387a.217.217 0 0 1 0-.412l1.162-.387a1.734 1.734 0 0 0 1.097-1.097l.387-1.162zM13.863.099a.145.145 0 0 1 .274 0l.258.774c.115.346.386.617.732.732l.774.258a.145.145 0 0 1 0 .274l-.774.258a1.156 1.156 0 0 0-.732.732l-.258.774a.145.145 0 0 1-.274 0l-.258-.774a1.156 1.156 0 0 0-.732-.732l-.774-.258a.145.145 0 0 1 0-.274l.774-.258c.346-.115.617-.386.732-.732L13.863.1z"></path></symbol><symbol id="sun-fill" viewbox="0 0 16 16"><path d="M8 12a4 4 0 1 0 0-8 4 4 0 0 0 0 8zM8 0a.5.5 0 0 1 .5.5v2a.5.5 0 0 1-1 0v-2A.5.5 0 0 1 8 0zm0 13a.5.5 0 0 1 .5.5v2a.5.5 0 0 1-1 0v-2A.5.5 0 0 1 8 13zm8-5a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1 0-1h2a.5.5 0 0 1 .5.5zM3 8a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1 0-1h2A.5.5 0 0 1 3 8zm10.657-5.657a.5.5 0 0 1 0 .707l-1.414 1.415a.5.5 0 1 1-.707-.708l1.414-1.414a.5.5 0 0 1 .707 0zm-9.193 9.193a.5.5 0 0 1 0 .707L3.05 13.657a.5.5 0 0 1-.707-.707l1.414-1.414a.5.5 0 0 1 .707 0zm9.193 2.121a.5.5 0 0 1-.707 0l-1.414-1.414a.5.5 0 0 1 .707-.707l1.414 1.414a.5.5 0 0 1 0 .707zM4.464 4.465a.5.5 0 0 1-.707 0L2.343 3.05a.5.5 0 1 1 .707-.707l1.414 1.414a.5.5 0 0 1 0 .708z"></path></symbol></svg><a class="visually-hidden-focusable skip-link" href="#main-content">Skip to main content</a>
  <div class="container-fluid top-nav-container">
    <div class="col-md-8">
      <div class="large-logo">
        <img id="incubator-logo" alt="Carpentries Incubator" src="../assets/images/incubator-logo.svg"><span class="badge text-bg-danger">
          <abbr title="This lesson is in the pre-alpha phase, which means that it is in early development, but has not yet been taught.">
            <a href="https://cdh.carpentries.org/the-lesson-life-cycle.html#early-development-pre-alpha-through-alpha" class="external-link alert-link">
              <i aria-hidden="true" class="icon" data-feather="alert-octagon" style="border-radius: 5px"></i>
              Pre-Alpha
            </a>
            <span class="visually-hidden">This lesson is in the pre-alpha phase, which means that it is in early development, but has not yet been taught.</span>
          </abbr>
        </span>

      </div>
    </div>
    <div class="selector-container">
      <div id="theme-selector">
        <li class="nav-item dropdown" id="theme-button-list">
          <button class="btn btn-link nav-link px-0 px-lg-2 dropdown-toggle d-flex align-items-center" id="bd-theme" type="button" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static" aria-label="Toggle theme (auto)">
            <svg class="bi my-1 theme-icon-active"><use href="#circle-half"></use></svg><i data-feather="chevron-down"></i>
          </button>
          <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="bd-theme-text">
<li>
              <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="light" aria-pressed="false">
                <svg class="bi me-2 theme-icon"><use href="#sun-fill"></use></svg>
                Light
                <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
            </li>
            <li>
              <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="dark" aria-pressed="false">
                <svg class="bi me-2 theme-icon"><use href="#moon-stars-fill"></use></svg>
                Dark
                <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
            </li>
            <li>
              <button type="button" class="btn dropdown-item d-flex align-items-center active" data-bs-theme-value="auto" aria-pressed="true">
                <svg class="bi me-2 theme-icon"><use href="#circle-half"></use></svg>
                Auto
                <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
            </li>
          </ul>
</li>
      </div>

      <div class="dropdown" id="instructor-dropdown">
        <button class="btn btn-secondary dropdown-toggle bordered-button" type="button" id="dropdownMenu1" data-bs-toggle="dropdown" aria-expanded="false">
          <i aria-hidden="true" class="icon" data-feather="eye"></i> Instructor View <i data-feather="chevron-down"></i>
        </button>
        <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
<li><button class="dropdown-item" type="button" onclick="window.location.href='../aio.html';">Learner View</button></li>
        </ul>
</div>
    </div>
  </div>
  <hr></header><nav class="navbar navbar-expand-xl bottom-nav incubator" aria-label="Main Navigation"><div class="container-fluid nav-container">
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle Navigation">
      <span class="navbar-toggler-icon"></span>
      <span class="menu-title">Menu</span>
    </button>
    <div class="nav-logo">
      <img class="small-logo" alt="Carpentries Incubator" src="../assets/images/incubator-logo-sm.svg">
</div>
    <div class="lesson-title-md">
      CRC1550 data management training material
    </div>
    <div class="search-icon-sm">
      <!-- TODO: do not show until we have search
        <i role="img" aria-label="Search the All In One page" data-feather="search"></i>
      -->
    </div>
    <div class="desktop-nav">
      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
<li class="nav-item">
          <span class="lesson-title">
            CRC1550 data management training material
          </span>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/key-points.html">Key Points</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/instructor-notes.html">Instructor Notes</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/images.html">Extract All Images</a>
        </li>
        <li class="nav-item dropdown">
          <button class="nav-link dropdown-toggle" id="navbarDropdown" data-bs-toggle="dropdown" aria-expanded="false">
            More <i data-feather="chevron-down"></i>
          </button>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
<hr>
<li><a class="dropdown-item" href="reference.html">Reference</a></li>
          </ul>
</li>
      </ul>
</div>
    <!--
    <form class="d-flex col-md-2 search-form">
      <fieldset disabled>
      <input class="form-control me-2 searchbox" type="search" placeholder="" aria-label="">
        <button class="btn btn-outline-success tablet-search-button"  type="submit">
          <i class="search-icon" data-feather="search" role="img" aria-label="Search the All In One page"></i>
        </button>
      </fieldset>
    </form>
    -->
    <a id="search-button" class="btn btn-primary" href="../instructor/aio.html" role="button" aria-label="Search the All In One page">Search the All In One page</a>
  </div>
<!--/div.container-fluid -->
</nav><div class="col-md-12 mobile-title">
  CRC1550 data management training material
</div>

<aside class="col-md-12 lesson-progress"><div style="width: %" class="percentage">
    %
  </div>
  <div class="progress incubator">
    <div class="progress-bar incubator" role="progressbar" style="width: %" aria-valuenow="" aria-label="Lesson Progress" aria-valuemin="0" aria-valuemax="100">
    </div>
  </div>
</aside><div class="container">
      <div class="row">
        <!-- START: inst/pkgdown/templates/navbar.html -->
<div id="sidebar-col" class="col-lg-4">
  <div id="sidebar" class="sidebar">
      <nav aria-labelledby="flush-headingEleven"><button role="button" aria-label="close menu" alt="close menu" aria-expanded="true" aria-controls="sidebar" class="collapse-toggle" data-collapse="Collapse " data-episodes="Episodes ">
          <i class="search-icon" data-feather="x" role="img"></i>
        </button>
        <div class="sidebar-inner">
          <div class="row mobile-row" id="theme-row-mobile">
            <div class="col" id="theme-selector">
              <li class="nav-item dropdown" id="theme-button-list">
                <button class="btn btn-link nav-link px-0 px-lg-2 dropdown-toggle d-flex align-items-center" id="bd-theme" type="button" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static" aria-label="Toggle theme (auto)">
                  <svg class="bi my-1 theme-icon-active"><use href="#circle-half"></use></svg><span class="d-lg-none ms-1" id="bd-theme-text">Toggle Theme</span>
                </button>
                <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="bd-theme-text">
<li>
                    <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="light" aria-pressed="false">
                      <svg class="bi me-2 theme-icon"><use href="#sun-fill"></use></svg>
                      Light
                      <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
                  </li>
                  <li>
                    <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="dark" aria-pressed="false">
                      <svg class="bi me-2 theme-icon"><use href="#moon-stars-fill"></use></svg>
                      Dark
                      <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
                  </li>
                  <li>
                    <button type="button" class="btn dropdown-item d-flex align-items-center active" data-bs-theme-value="auto" aria-pressed="true">
                      <svg class="bi me-2 theme-icon"><use href="#circle-half"></use></svg>
                      Auto
                      <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
                  </li>
                </ul>
</li>
            </div>
          </div>
          <div class="row mobile-row">
            <div class="col">
              <div class="sidenav-view-selector">
                <div class="accordion accordion-flush" id="accordionFlush9">
                  <div class="accordion-item">
                    <h2 class="accordion-header" id="flush-headingNine">
                      <button class="accordion-button collapsed" id="instructor" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseNine" aria-expanded="false" aria-controls="flush-collapseNine">
                        <i id="eye" aria-hidden="true" class="icon" data-feather="eye"></i> Instructor View
                      </button>
                    </h2>
                    <div id="flush-collapseNine" class="accordion-collapse collapse" aria-labelledby="flush-headingNine" data-bs-parent="#accordionFlush2">
                      <div class="accordion-body">
                        <a href="../aio.html">Learner View</a>
                      </div>
                    </div>
                  </div>
<!--/div.accordion-item-->
                </div>
<!--/div.accordion-flush-->
              </div>
<!--div.sidenav-view-selector -->
            </div>
<!--/div.col -->

            <hr>
</div>
<!--/div.mobile-row -->

          <div class="accordion accordion-flush" id="accordionFlush11">
            <div class="accordion-item">

              <button id="chapters" class="accordion-button show" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseEleven" aria-expanded="false" aria-controls="flush-collapseEleven">
                <h2 class="accordion-header chapters" id="flush-headingEleven">
                  EPISODES
                </h2>
              </button>
              <div id="flush-collapseEleven" class="accordion-collapse show collapse" aria-labelledby="flush-headingEleven" data-bs-parent="#accordionFlush11">

                <div class="accordion-body">
                  <div class="accordion accordion-flush" id="accordionFlush1">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading1">
        <a href="index.html">Summary and Schedule</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush2">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading2">
        <a href="introduction.html">1. Preamble to Data Management Plan</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush3">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading3">
        <a href="bionfo_primer.html">2. An introduction to bioinformatics file formats.</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush4">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading4">
        <a href="fair-principles.html">3. FAIR Principles in OMICs Research</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush5">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading5">
        <a href="sodar.html">4. SODAR: System for Omics Data Access and Retrieval.</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush6">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading6">
        <a href="sra.html">5. SRA</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

                </div>
              </div>
            </div>

            <hr class="half-width">
<div class="accordion accordion-flush lesson-resources" id="accordionFlush12">
              <div class="accordion-item">
                <h2 class="accordion-header" id="flush-headingTwelve">
                  <button class="accordion-button collapsed" id="lesson-resources" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseTwelve" aria-expanded="false" aria-controls="flush-collapseTwelve">
                    RESOURCES
                  </button>
                </h2>
                <div id="flush-collapseTwelve" class="accordion-collapse collapse" aria-labelledby="flush-headingTwelve" data-bs-parent="#accordionFlush12">
                  <div class="accordion-body">
                    <ul>
<li>
                        <a href="../instructor/key-points.html">Key Points</a>
                      </li>
                      <li>
                        <a href="../instructor/instructor-notes.html">Instructor Notes</a>
                      </li>
                      <li>
                        <a href="../instructor/images.html">Extract All Images</a>
                      </li>
                      <hr>
<li><a class="dropdown-item" href="reference.html">Reference</a></li>
                    </ul>
</div>
                </div>
              </div>
            </div>
            <hr class="half-width lesson-resources">
<a href="../instructor/aio.html">See all in one page</a>


            <hr class="d-none d-sm-block d-md-none">
<div class="d-grid gap-1">

            </div>
          </div>
<!-- /div.accordion -->
        </div>
<!-- /div.sidebar-inner -->
      </nav>
</div>
<!-- /div.sidebar -->
  </div>
<!-- /div.sidebar-col -->
<!-- END:   inst/pkgdown/templates/navbar.html-->

        <!-- START: inst/pkgdown/templates/content-extra.html -->
  <div class="col-xl-8 col-lg-12 primary-content">
    <main id="main-content" class="main-content"><div class="container lesson-content">
        
        
<section id="aio-introduction"><p>Content from <a href="introduction.html">Preamble to Data Management Plan</a></p>
<hr>
<p>Last updated on 2024-10-08 |

        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/episodes/introduction.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 0 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>What is Metadata?</li>
<li>How to describe metadata?</li>
<li>What is the difference between data and metadata?</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Defining data management plan.</li>
<li>Understanding its importance applied to OMICs experiments.</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section><h2 class="section-heading" id="introduction">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h2>
<hr class="half-width">
<p>In this lesson, we will learn about metadata and its importance. We
will learn about the different types of metadata and how it is used to
provide context and meaning to data. We will also learn about the
importance of metadata standardization and how it can help to improve
data management and data analysis. Finally, we will learn about the
importance of metadata in OMICs experiments and how it can help to
improve the reproducibility and reliability of research results.</p>
<div class="section level3">
<h3 id="what-is-metadata">What is Metadata?<a class="anchor" aria-label="anchor" href="#what-is-metadata"></a>
</h3>
<p>Metadata is data that provides information about other data. It is an
essential component of any data analysis pipeline. It is used to
describe the content, quality, condition, and other characteristics of
data. Metadata is used to provide context and meaning to data. It is
used to help users understand the data and to help them make informed
decisions about how to use it.</p>
</div>
<div class="section level3">
<h3 id="data-vs--metadata">Data vs. Metadata<a class="anchor" aria-label="anchor" href="#data-vs--metadata"></a>
</h3>
<p>Data is the raw information that is collected and analyzed in a
research study. It is the information that is used to answer research
questions and to make decisions. Metadata is the information that
describes the data. It is the information that provides context and
meaning to the data. It is used to help users understand the data and to
help them make informed decisions about how to use it.</p>
<p>An everyday analogy is digital photography. Every time you take a
photo with a digital camera, a range of “behind-the-scenes” information
is saved along with the image data. This is known as metadata, and it
can include details of the camera, lens, and shooting settings used,
plus optional information about the photographer, location and more.
Similarly, every experiment generates data, and contains metadata.</p>
<p>An experimental protocol, for example, is considered part of the
metadata of an experiment. There are different levels of metadata, which
will be discussed in more detail once we discuss the ISA (Investigation,
Study and Assay) framework.</p>
</div>
</section><section><h2 class="section-heading" id="data-management-plan">Data management plan<a class="anchor" aria-label="anchor" href="#data-management-plan"></a>
</h2>
<hr class="half-width">
<p>A data management plan (DMP) is a document that outlines how data
will be handled throughout a research project. It explains how to
collect, organize, store, and share data, as well as ways to make sure
that data is safe and meets legal and ethical standards. The DMP aims to
enhance the accessibility, usability, and reproducibility of the data,
as detailed below.</p>
<div class="section level3">
<h3 id="goals-of-data-management-plan">Goals of data management plan<a class="anchor" aria-label="anchor" href="#goals-of-data-management-plan"></a>
</h3>
<p>Here we define the goals of a data management plan in context to the
FAIR framework:</p>
<ul>
<li>Automated deposition of data and metadata to public repositories
(findability).</li>
<li>Ensure that data is stored in a format that is easily accessible and
usable (accessibility).</li>
<li>Define consistent terminologies and controlled vocabularies to
represent the data (interoperability).</li>
<li>Ensure that data is stored in a secure and reliable manner
(reusability).</li>
</ul>
</div>
<div class="section level3">
<h3 id="roles-and-responsibilities">Roles and responsibilities<a class="anchor" aria-label="anchor" href="#roles-and-responsibilities"></a>
</h3>
<ul>
<li>Data owners: Persons responsible for generating the data and
metadata.</li>
<li>Data managers: The person responsible for managing the data and
metadata.</li>
<li>Data users: The person responsible for using the data and metadata
to answer research questions and make decisions.</li>
</ul>
</div>
<div class="section level3">
<h3 id="benefits-of-data-management-and-metadata-standardization">Benefits of Data Management and Metadata Standardization<a class="anchor" aria-label="anchor" href="#benefits-of-data-management-and-metadata-standardization"></a>
</h3>
<ul>
<li>Increase impact and visibility of research.</li>
<li>Assigns clear responsibilities for data management tasks such as
storage, backup, and controlled sharing.</li>
<li>Standardizes data and metadata to facilitate collaboration using
consistent terminologies and controlled vocabularies.</li>
<li>Promotes collaboration within the CRC and with external partners,
supporting method development and data reuse.</li>
<li>Ensures compliance with funder requirements.</li>
<li>Facilitates efficient interoperability among infrastructure
teams.</li>
<li>Enable audit trail of data and metadata changes.</li>
</ul>
</div>
</section><section><h2 class="section-heading" id="metadata-management-plan-and-fair-framewok">Metadata management plan and FAIR framewok<a class="anchor" aria-label="anchor" href="#metadata-management-plan-and-fair-framewok"></a>
</h2>
<hr class="half-width">
<p>In this document we outline the strategies and procedures for
managing high-throughput data within the CRC1550’s integrated
infrastructure. The CRC-INF plan is centered on the consolidation of
interfaces, procedures, and interoperability standards (Aim 1) and the
development of generic bioinformatics workflows for the standard
analysis of multimodal data sets from CRC projects (Aim 2). The primary
objective of the data management team is to ensure the efficient
management of unprocessed and processed data produced by high-throughput
experiments by providing services (and interfaces) to manage and store
data and metadata.</p>
<figure><img src="https://lh7-us.googleusercontent.com/HIZrk0S_WDLJXpKYFcl6BfZY5or_grINIHr6JNW_5mZmpb3xLxy6_aQEcPiEAmGwL_pS6LpaJ9aPpHUEbDZ-DSDUpPtE_cciZNdvwZGSsqYUJ9br8Bzr_bacQ2UdG1-WFKlsMBUu9LBpergxc4EtppU" alt="The CRC1550-INF working program" class="figure mx-auto d-block"><div class="figcaption">The CRC1550-INF working program</div>
</figure><p>There are three working groups:</p>
<ul>
<li><p>The BioBanking working group is responsible for the collection,
storage, and management of biological samples.</p></li>
<li><p>The Data Management working group.</p></li>
<li><p>The bioinformatics working group is responsible for the analysis
of high-throughput data.</p></li>
</ul>
<p>This course is focused on the Data Management working group.</p>
<div class="section level3">
<h3 id="findability">Findability<a class="anchor" aria-label="anchor" href="#findability"></a>
</h3>
<p>Metadata and data should be easily discoverable by both humans and
machines. To ensure findability, each data object must have a unique and
persistent identifier: - Data objects must be re-findable at any time,
with an emphasis on maintaining persistent metadata. - Each data object
should include basic machine-readable metadata to differentiate it from
other objects. - Identifiers for concepts used in data objects must be
unique and persistent.</p>
</div>
<div class="section level3">
<h3 id="accessibility">Accessibility<a class="anchor" aria-label="anchor" href="#accessibility"></a>
</h3>
<p>After finding data, it’s crucial to understand the conditions for
accessing them. Data should be accessible to both machines and
humans:</p>
<ul>
<li>With proper authorization.</li>
<li>Via a well-defined protocol.</li>
</ul>
<p>This ensures that both machines and humans can determine the actual
accessibility of each data object.</p>
</div>
<div class="section level3">
<h3 id="interoperability">Interoperability<a class="anchor" aria-label="anchor" href="#interoperability"></a>
</h3>
<p>Data needs to be processed and, often, be integrated with other data
sets and be compatible with various applications or workflows for
analysis, storage, and processing.</p>
<p>Data objects are interoperable if: - The (meta)data is
machine-actionable. - The (meta)data formats use shared vocabularies or
ontologies. - The (meta)data within the data object is both
syntactically parsable and semantically machine-accessible.</p>
</div>
<div class="section level3">
<h3 id="reusability">Reusability<a class="anchor" aria-label="anchor" href="#reusability"></a>
</h3>
<p>The ultimate goal is to optimize the reuse of data, requiring that
metadata and data be well-described for use, replication, and
combination in various settings.</p>
<p>For data objects to be reusable, they must:</p>
<ul>
<li>Comply with principles of findability, accessibility, and
interoperability.</li>
<li>Be sufficiently well-described and rich in metadata to allow
automatic linking or integration with relevant data sources, with
minimal human effort.</li>
<li>Include rich metadata and provenance information for proper citation
of published data objects and their sources.</li>
</ul>
</div>
</section><section><h2 class="section-heading" id="conclusions">Conclusions<a class="anchor" aria-label="anchor" href="#conclusions"></a>
</h2>
<hr class="half-width">
<p>Data and metadata standardization and description involves
establishing and applying rules and guidelines for creating, managing,
and using metadata. This process ensures that metadata is reliable and
helps users more effectively find, access, and utilize data.</p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul>
<li>Data management plan execution enables data FAIRification.</li>
<li>Metadata standardization is critical for reliable results and
reproducibility.</li>
<li>There are three roles in data management: data owners, managers, and
users.</li>
<li>Data management and metadata standardization provide numerous
benefits that enhance the overall quality, usability, and impact of
research data.</li>
</ul>
</div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</section></section><section id="aio-bionfo_primer"><p>Content from <a href="bionfo_primer.html">An introduction to bioinformatics file formats.</a></p>
<hr>
<p>Last updated on 2024-10-08 |

        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/episodes/bionfo_primer.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 0 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>What are the different files formats used in bioinformatics?</li>
<li>Differences between raw and processed data sets.</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Introduce the idea of OMICs experiments.</li>
<li>Familiarize with commonly used file formats and concepts.</li>
<li>Understanding the multilayered nature of metadata from OMICs
experiments.</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section><h2 class="section-heading" id="introduction">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h2>
<hr class="half-width">
<p>OMICs technologies provide unbiased measurements of molecular
entities in biological samples. These measurements allow for a
comprehensive exploration of functions and association of such entities.
System biology is a branch of biology that aims to comprehend biological
systems in their entirety by the application of OMICs methods.</p>
<div id="crc1550-focus" class="callout">
<div class="callout-square">
<i class="callout-icon" data-feather="bell"></i>
</div>
<div id="crc1550-focus" class="callout-inner">
<h3 class="callout-title">CRC1550 focus</h3>
<div class="callout-content">
<p>Within the CRC1550, the data management plan currently focuses on
animal experiments, and the data generated from specific OMICs
experiments. The data management plan is a living document that will be
updated as the project progresses. Currently, we don’t consider data
human data from patient samples, such as clinical data, or imaging data,
which are current out of the scope of this material.</p>
</div>
</div>
</div>
<p><strong>OMICs</strong> is a collective term for a group of
technologies that allow for the comprehensive analysis of biological
molecules. These technologies include genomics, transcriptomics,
proteomics, metabolomics, and lipidomics. Each of these technologies
provides a unique perspective on the molecular composition of a
biological sample, and together they allow for a comprehensive
exploration of the functions and associations of biological
molecules.</p>
</section><section><h2 class="section-heading" id="general-definition">General definition<a class="anchor" aria-label="anchor" href="#general-definition"></a>
</h2>
<hr class="half-width">
<ul>
<li>
<strong>Genomics</strong>: Study of an organism’s entire genome,
exome or selected regions, offering insights into genetic variations.
When associated with disease phenotype or clinical data can be
implicated on health and disease.</li>
<li>
<strong>Transcriptomics</strong>: Focuses on the transcriptome, the
complete set of RNA transcripts produced by the genome, under specific
circumstances or in a specific cell.</li>
<li>
<strong>Proteomics</strong>: Examines the proteome, the entire set
of proteins produced or modified by an organism, revealing functions and
pathways.</li>
<li>
<strong>Metabolomics</strong>: Involves the study of metabolites,
providing a snapshot of the physiological condition of a cell or
organism.</li>
<li>
<strong>Lipidomics</strong>: A complete lipid profile within a cell,
tissue, or organism, allowing the study of their roles in cellular
processes.</li>
</ul></section><section><h2 class="section-heading" id="file-formats">File formats<a class="anchor" aria-label="anchor" href="#file-formats"></a>
</h2>
<hr class="half-width">
<p>There are a multitude of file formats used in bioinformatics, each
with its specific use case. Some of the most commonly used file formats
include:</p>
<p><strong>FASTQ format</strong>: a text-based format for storing both
oligonucleotide sequences and its corresponding quality scores. Obtained
after base calling procedure.</p>
<div id="fastq-file-format" class="callout">
<div class="callout-square">
<i class="callout-icon" data-feather="bell"></i>
</div>
<div id="fastq-file-format" class="callout-inner">
<h3 class="callout-title">FASTQ file format</h3>
<div class="callout-content">
<p>For bioinformatics workflow, the FASTQ format is often the initial
step. This file format stores “reads”, or nucleotide sequences, along
with their quality scores. Each sequence in a FASTQ file is represented
by four lines: a header with a ‘@’ symbol followed by an identifier and
optional description, the nucleotide sequence itself, a separator line
starting with a ‘+’, and the quality scores encoded as ASCII characters.
This structure allows for efficient storage and analysis of sequencing
data, essential for tasks like genome assembly and analysis. For more
detailed information, please visit the Wikipedia page on <a href="https://en.wikipedia.org/wiki/FASTQ_format" class="external-link">FASTQ format</a>.</p>
</div>
</div>
</div>
<p><strong>VCF/BCF</strong>: Stores genetic variation data, such as
single nucleotide polymorphisms (SNPs) and insertions/deletions
(indels).</p>
<p><strong>SAM/BAM/CRAM files</strong>: These are used to store sequence
alignment data, and are used to represent the alignment of reads to a
reference genome. SAM is a text-based format, while BAM is a binary
version of the same format. CRAM is a compressed version of the BAM
format.</p>
<p><strong>PRIDE XML, mzIdentML, mzTab, and mzML</strong>: These file
formats are used to store, process, and visualize mass
spectrometry-based proteomics data deposited in the PRIDE Archive.</p>
<ul>
<li>PRIDE XML: The internal data format and submission format for the
PRIDE database, capturing comprehensive metadata and experimental
details.</li>
<li>mzIdentML: A standardized format for reporting peptide and protein
identifications, including search results and confidence metrics.</li>
<li>mzTab: A simpler, tab-delimited format for reporting
identifications, quantification results, and metadata, designed for ease
of use and flexibility.</li>
<li>mzML: An XML-based open standard for storing raw mass spectrometry
data, encompassing detailed spectral information and metadata.</li>
</ul>
<div class="section level3">
<h3 id="types-of-file-formats">Types of file formats<a class="anchor" aria-label="anchor" href="#types-of-file-formats"></a>
</h3>
<p><strong>Proprietary formats:</strong> high-throughput machine such as
Illumina, Nanopore, produce files before base calling. These files are
in the form of binary files, and are not human-readable and depend on
vendor-specific methods for signal processing.</p>
<p>Note that various proprietary formats are used for raw proteomics
data depending on the equipment vendor.</p>
<p><strong>Processed file formats:</strong> formats whom data has been
processed and are ready for downstream analysis. These formats are often
tabular, and can be easily imported into R or Python for further
analysis. Examples include:</p>
<ul>
<li><p><strong>Tabular formats</strong>: csv, xlsx, tsv, and
others.</p></li>
<li><p><strong>RData</strong>: R-specific binary format for storing R
objects.</p></li>
<li><p><strong>HDF5</strong>: A data model, library, and file format for
storing and managing large and complex data.</p></li>
<li><p><strong>BED, GFF, and GTF</strong>: These are used to store
genomic feature or annotations, such as gene locations, and
structure.</p></li>
</ul>
<p>The scope of this lesson is to give an overview of file formats used
in bioinformatics. It is important to understand that the different
files a specific purpose, can be vendor-specific and that the choice of
file format can have a significant impact on the downstream analysis of
the data. For example, use of legacy vendor-specific format can limit
the use of the data in modern bioinformatics pipelines, as proprietary
programs may disappear. In general, it’s good practice to use
open-source file formats.</p>
<div id="raw-vs-processed-data" class="callout">
<div class="callout-square">
<i class="callout-icon" data-feather="bell"></i>
</div>
<div id="raw-vs-processed-data" class="callout-inner">
<h3 class="callout-title">Raw vs processed data</h3>
<div class="callout-content">
<p>As a general rule of thumb, one can consider the raw data as the data
that comes directly from the sequencing machine, and the processed data
any data set derived from the raw data. The raw data is often in a
format that is not human-readable, and requires specialized software to
process. This definition may change depending on context.</p>
</div>
</div>
</div>
</div>
</section><section><h2 class="section-heading" id="challenges-in-omics-data">Challenges in OMICs data:<a class="anchor" aria-label="anchor" href="#challenges-in-omics-data"></a>
</h2>
<hr class="half-width">
<p>There are many challenges involved in the use of OMICs datasets.</p>
<ul>
<li>
<strong>Data storage</strong>: OMICs data sets can be large, and
require robust data storage and backup solutions. This is especially
true for raw sequencing data, which can be many gigabytes in size. We
expect more than 20 terabytes of high-throughput data to be produce per
year.</li>
<li>
<strong>Data analysis</strong>: OMICs data sets are complex, and
require specialized methods for analysis.</li>
<li>
<strong>Standardization</strong>: As there are many file formats,
there are many data standards and protocols used in OMICs research. This
lack of standardization make it difficult to compare data between
different centres.</li>
<li>
<strong>Data sharing</strong>: OMICs data sets are required to be
shared upon publication.</li>
</ul>
<p>All the challenges can be mitigated through the use of the meta data
managing plan. Metadata is essential for understanding the context of
the data, and is used to help find and interpret the data. In the
context of OMICs data, metadata can include information about the sample
donor, sample, the experimental conditions, the data collection methods,
protocols and the data analysis methods. Results files can also contain
metadata describing the processing steps used on the raw files. Metadata
is essential for reproducibility, as it allows others to understand and
replicate the results of experiments.</p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul>
<li>
<strong>OMICs technologies</strong> are crucial for a holistic
understanding of biological systems.</li>
<li>Common file formats in bioinformatics serve specific purposes and
require specialized toolsets.</li>
<li>Metadata plays a critical role in contextualizing and modeling OMICs
data.</li>
</ul>
</div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</section></section><section id="aio-fair-principles"><p>Content from <a href="fair-principles.html">FAIR Principles in OMICs Research</a></p>
<hr>
<p>Last updated on 2024-10-08 |

        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/episodes/fair-principles.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 20 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>What actions can we take adhere to FAIR framework?</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Understand the FAIR framework applied to OMICs research</li>
<li>Discuss solution the INF data management team can provide so the
project adhere to the FAIR framework</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<div class="section level1">
<h1 id="reminder-of-fair-framework">Reminder of FAIR framework<a class="anchor" aria-label="anchor" href="#reminder-of-fair-framework"></a>
</h1>
<p>This chapter aims to give a deep dive on FAIR framework.</p>
<ul>
<li>
<strong>Findable</strong>: Metadata helps in uniquely identifying
data for easy discovery.</li>
<li>
<strong>Accessible</strong>: Metadata includes information on how
data can be accessed.</li>
<li>
<strong>Interoperable</strong>: Standardized metadata ensures that
data can be integrated with other datasets.</li>
<li>
<strong>Reusable</strong>: Metadata provides the information
necessary for data to be reused effectively, including the conditions
under which it can be reused.</li>
</ul>
<div class="section level2">
<h2 id="findable">Findable<a class="anchor" aria-label="anchor" href="#findable"></a>
</h2>
<p>The “Findable” aspect of the FAIR principles emphasizes the need for
data to be easily located and identified. This is the first step in
ensuring that data can be used and reused effectively. Metadata plays a
crucial role in making data findable by providing information about the
data, such as its title, publication, author, and description. This
information helps users to locate and identify the data they need for
their research. Or describing sample location (left-ventricle vs. right
ventricle), library preparation type (Poly-A enrichment vs. Ribo
depletion).</p>
<p>Another important aspect for findability is the use of persistent
identifier. One every day aspect is nomenclature used for genes or
proteins in manuscripts. While the use of nomenclature standards
advanced, use of common names can lead to ambiguities or confusion. In
that case, the use of persistent identifier, such as the <a href="http://www.ensembl.org/info/genome/stable_ids/index.html" class="external-link">Ensembl’s
Stable IDs</a> or <a href="https://www.uniprot.org/help/accession_numbers" class="external-link">UniProt accession
numbers</a> are preferred. The Digital Object identifier (<a href="https://dx.doi.org/" class="external-link">DOI</a>) is used to make data, software, and
other research outputs easily citable and trackable.</p>
<p>Metadata can also facilitate data discovery. For example, metadata
can be used on</p>
</div>
<div class="section level2">
<h2 id="accessible">Accessible<a class="anchor" aria-label="anchor" href="#accessible"></a>
</h2>
<p>Making data accessible to collaborators and the public increases its
impact. Our INF aims to facilitate and automatize the limited or public
distribution of data sets. The critical point is establishing user
access with granularity.</p>
<p>The Member Roles in SODAR are structured to provide varying levels of
access and functionality within projects. Each user can hold only one
role at a time within a category or project, with “project” serving as a
generic term encompassing both categories and specific projects. The
roles, listed in descending order of rights and functionality,
include:</p>
<ol style="list-style-type: decimal">
<li><p><strong>Project Owner</strong>: Holds full access to project data
and functionality, with the ability to assign roles, including
delegates. They can transfer ownership to another user.</p></li>
<li><p><strong>Project Delegate</strong>: Possesses full access to
project data and functionality, except for modifying owner or delegate
roles. This role can only be assigned by a project owner.</p></li>
<li><p><strong>Project Contributor</strong>: Granted access to create
and modify data within a project, such as uploading files and editing
sample sheets, with certain limitations. They cannot modify project
metadata or user roles.</p></li>
<li><p><strong>Project Guest</strong>: Limited to read-only access to
project data.</p></li>
<li><p><strong>Project Finder</strong>: Exclusive to users in
categories, allowing them to view child categories and projects, along
with member lists, without accessing project data or apps. This role is
useful for staff members who need an overview of the category and
project structure.</p></li>
</ol>
<p>Role inheritance is implemented, with roles inherited from parent
categories. These inherited roles can be promoted for the current
category or project but cannot be demoted. Inherited roles are denoted
in the member list, indicating the category from which the role is
inherited.</p>
<p>Additionally, each project must have one local owner who is not
inherited.</p>
<p>Moreover, standardized metadata can facilitate data users to find
your data set on specialized search engines, such as <a href="https://www.ncbi.nlm.nih.gov/sra" class="external-link">Sequence Read Archive</a>, <a href="https://www.ebi.ac.uk/ena/browser/" class="external-link">European Nucleotide
Archive</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/" class="external-link">Gene
Expression Omnibus</a>, or other engines, such as the <a href="https://metasra.biostat.wisc.edu/" class="external-link">METASra</a> or the <a href="https://sra-explorer.info/" class="external-link">SRA explorer</a>. The more data users
find your data set, the more likely it receives citations.</p>
</div>
<div class="section level2">
<h2 id="interoperable">Interoperable<a class="anchor" aria-label="anchor" href="#interoperable"></a>
</h2>
<p>Data interoperability refers to the ability of different systems,
organizations, and applications to access, exchange, and use data
seamlessly and effectively. It ensures that data from diverse sources
can be integrated and utilized cohesively, regardless of the differences
in their formats, structures, or origins. For data to be interoperable,
it must meet the following key criteria:</p>
<ol style="list-style-type: decimal">
<li>Machine-readable.</li>
<li>Standardized formats and processing steps.</li>
<li>Using common vocabularies and ontologies.</li>
</ol>
<p>Adhering to these criteria facilitates data use and integration, as
well further analysis, such meta analysis or database inclusion. The
main action here is selecting a common ontology to describe experiments.
Please check this resource to see how ontologies work: <a href="https://www.ebi.ac.uk/ols4/search?q=left+ventricle" class="external-link">ols4 left
ventricle query</a>.</p>
<figure><img src="../fig/OLS4_lv_query.png" alt="OLS4 query for left ventricle." class="figure mx-auto d-block"><div class="figcaption">OLS4 query for left ventricle.</div>
</figure>
</div>
<div class="section level2">
<h2 id="reusable">Reusable<a class="anchor" aria-label="anchor" href="#reusable"></a>
</h2>
<p>Data set reusability is the final goal of the FAIR framework.
Reusability requires adhering to four key concepts:</p>
<ul>
<li>The data complies with the Findable, Accessible and Interoperable
principles.</li>
<li>Data and metadata can be validated.</li>
<li>Data permission and licensing.</li>
</ul>
<div class="section level3">
<h3 id="data-validation">Data validation<a class="anchor" aria-label="anchor" href="#data-validation"></a>
</h3>
<p>Another important concept is providing standardized file formats and
validating such files. Using MD5 checksums for file sharing and
validation is important because it ensures data integrity. Before a file
is shared, an MD5 checksum should be generated, which is a unique hash
value representing the file’s content. The recipient can generate their
own MD5 checksum for the received file and compare it with the original
checksum. If the values match, it confirms that the file has not been
altered or corrupted during transmission, ensuring the file’s integrity
and authenticity.</p>
<p>Here is an example of how to test a file checksum manually:</p>
<div class="codewrapper sourceCode" id="cb1">
<h3 class="code-label">BASH<i aria-hidden="true" data-feather="chevron-left"></i><i aria-hidden="true" data-feather="chevron-right"></i>
</h3>
<pre class="sourceCode bash" tabindex="0"><code class="sourceCode bash"><span id="cb1-1"><a href="#cb1-1" tabindex="-1"></a><span class="bu">echo</span> <span class="st">"this is a text file."</span> <span class="op">&gt;</span> test_checksum.txt</span>
<span id="cb1-2"><a href="#cb1-2" tabindex="-1"></a><span class="fu">cat</span> test_checksum.txt</span>
<span id="cb1-3"><a href="#cb1-3" tabindex="-1"></a><span class="op">&gt;</span> this <span class="ex">is</span> a text file.</span>
<span id="cb1-4"><a href="#cb1-4" tabindex="-1"></a><span class="fu">md5sum</span> test_checksum.txt <span class="op">&gt;</span> test_checksum.md5</span>
<span id="cb1-5"><a href="#cb1-5" tabindex="-1"></a><span class="fu">cat</span> test_checksum.md5</span>
<span id="cb1-6"><a href="#cb1-6" tabindex="-1"></a><span class="op">&gt;</span> bc8a22e96d2b46f583a8c5fb055de679  <span class="ex">test_checksum</span></span>
<span id="cb1-7"><a href="#cb1-7" tabindex="-1"></a><span class="fu">md5sum</span> <span class="at">--check</span> test_checksum.md5</span>
<span id="cb1-8"><a href="#cb1-8" tabindex="-1"></a><span class="op">&gt;</span>test_checksum: <span class="ex">OK</span></span></code></pre>
</div>
<div class="codewrapper">
<h3 class="code-label">OUTPUT<i aria-hidden="true" data-feather="chevron-left"></i><i aria-hidden="true" data-feather="chevron-right"></i>
</h3>
<pre class="output" tabindex="0"><code>Error in running command bash</code></pre>
</div>
</div>
<div class="section level3">
<h3 id="metadata-validation">Metadata validation<a class="anchor" aria-label="anchor" href="#metadata-validation"></a>
</h3>
<p>There are specialized tools for validating metadata. SODAR uses <a href="https://github.com/bihealth/altamisa" class="external-link uri">altamisa</a>.
SODAR user interface will display errors and warning in case of parsing
errors for the metadata files.</p>
</div>
<div class="section level3">
<h3 id="data-licenses">Data licenses<a class="anchor" aria-label="anchor" href="#data-licenses"></a>
</h3>
<div id="disclaimer" class="callout">
<div class="callout-square">
<i class="callout-icon" data-feather="bell"></i>
</div>
<div id="disclaimer" class="callout-inner">
<h3 class="callout-title">Disclaimer</h3>
<div class="callout-content">
<p>This section does not aim to discuss legal aspects of data sharing,
but to provide an overview of the importance of data licenses. For legal
advice, please consult a legal expert.</p>
</div>
</div>
</div>
<p>Data licenses are legal instruments that define the terms and
conditions under which data can be used, shared, and distributed. They
provide a framework for data users to understand their rights and
obligations when using data, including the permissions granted by the
data provider and any restrictions on data use.</p>
<p>In legal terms, data and software can only be re-used if it licensed.
Data deposited to the SRA (Sequence Read Archive) is licensed under the
<a href="https://www.ncbi.nlm.nih.gov/home/about/policies/" class="external-link">NCBI data
use policy</a>. This license provides open access to raw sequencing
data, including RNA-seq data, under the public domain license. This
means the data can be freely used, modified, and shared by anyone for
any purpose without restrictions.</p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul>
<li>The FAIR framework is central for data and metadata management.</li>
<li>Data sets can be made more reusable by taking simple actions.</li>
</ul>
</div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</div>
</div>
</div></section><section id="aio-sodar"><p>Content from <a href="sodar.html">SODAR: System for Omics Data Access and Retrieval.</a></p>
<hr>
<p>Last updated on 2024-10-08 |

        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/episodes/sodar.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 30 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>How to use SODAR for data and metadata entry?</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Introduces SODAR and overview of its building blocks.</li>
<li>How the process for data and metadata upload works within the
CRC1550-INF.</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<div class="section level1">
<h1 id="overview">Overview<a class="anchor" aria-label="anchor" href="#overview"></a>
</h1>
<figure><img src="../fig/SodarLogo.svg" class="figure mx-auto d-block"></figure><p><strong>S</strong>ystem for <strong>O</strong>mics
<strong>D</strong>ata <strong>A</strong>ccess and
<strong>R</strong>etrieval (<strong>SODAR</strong>) is a specialized
system designed for managing data in OMICs research projects. The
platform provides researchers with a central location for navigating and
coordinating data, connecting to other resources and systems. Key
features of SODAR include project-based access control, modeling study
design metadata, large-scale data storage, file validation, and various
tools for data management. SODAR aims to address challenges faced by
scientists working on OMICs studies by offering a user-friendly
interface for managing multi-assay studies and organizing raw data set,
metadata and results in a single place. Most end users will use SODAR
via its web-based GUI, while data stewards will use the REST APIs. SODAR
server runs on the Django web server, models metadata with the ISA
specification, and uses iRODS for file storage. SODAR is developed by
the <a href="https://www.cubi.bihealth.org/" class="external-link">Core Unit
Bioinformatics</a> at the <a href="https://www.bihealth.org/" class="external-link">Berlin
Institute of Health</a>. The software is written in Python 3 and freely
available under the MIT license.</p>
</div>
<div class="section level1">
<h1 id="sodar-components-and-data-workflow">SODAR components and data workflow<a class="anchor" aria-label="anchor" href="#sodar-components-and-data-workflow"></a>
</h1>
<figure><img src="../fig/clipboard-1870335311.png" alt="Figure 1. Overview of data workflow with SODAR. The next sections will detail each component." class="figure mx-auto d-block"><div class="figcaption">
<strong>Figure 1.</strong> Overview of data
workflow with SODAR. The next sections will detail each component.</div>
</figure>
</div>
<div class="section level1">
<h1 id="sodar-server">SODAR server<a class="anchor" aria-label="anchor" href="#sodar-server"></a>
</h1>
<p>The SODAR server is the main component of the system. It comprises
several modules called Django apps, which are responsible for different
tasks. Although these apps are not the utmost importance for the users,
understanding the available apps allows users to understand the full
potential of the system. The main apps are detailed in the <a href="https://github.com/bihealth/sodar-server?tab=readme-ov-file#technical-information" class="external-link">GitHub
repository</a> of the project and listed below:</p>
<ul>
<li>
<strong>Samplesheets</strong>: Modeling of study metadata in the
ISA-Tab format</li>
<li>
<strong>Landingzones</strong>: Management of file validation and
uploads into iRODS</li>
<li>
<strong>Irodsadmin</strong>: iRODS data administration helpers</li>
<li>
<strong>Irodsbackend</strong>: Backend app for iRODS queries and
operations</li>
<li>
<strong>Irodsinfo</strong>: Display iRODS server information and
create user configurations</li>
<li>
<strong>Ontologyaccess</strong>: Parse, store and serve ontologies
for local lookup</li>
<li>
<strong>Taskflowbackend</strong>: Run iRODS transactions with full
rollback for project and file operations</li>
</ul>
<p>These apps are presented to end-users via the SODAR
web-application:</p>
<figure><img src="../fig/clipboard-3240582884.png" alt="Figure 2. SODAR developers provide a demo instance at this url." class="figure mx-auto d-block"><div class="figcaption">
<strong>Figure 2.</strong> SODAR developers
provide a demo instance at <a href="https://sodar-demo.cubi.bihealth.org/samplesheets/a1e84b38-568d-44e7-b81f-43ae81927187#/study/473c5d99-532b-4c31-a056-8e58d16a6bb7" class="external-link">this
url</a>.</div>
</figure>
</div>
<div class="section level1">
<h1 id="irods-server">iRODS server<a class="anchor" aria-label="anchor" href="#irods-server"></a>
</h1>
<p>iRODS stands for Integrated Rule-Oriented Data System, is the data
management system that powers SODAR data and metadata management
under-the-hood. Most user will not need to interact with iRODS directly.
iRODS provides storage virtualization among heterogeneous file systems,
meaning it allows data to be stored in different computers or external
servers (‘cloud’) and still be systematic organized in terms of
metadata. The usual use case is for organizing unstructured data, such
OMICs experiments.</p>
<p>The iCAT server serves as a metadata catalog that supports large
facility experimental data, linking all aspects of the research chain
from proposal through to publication. It utilizes a relational database
for storing metadata in the form of “triples,” consisting of an
attribute field, a value field, and a unit field.</p>
<p>An <strong>iRODS collection</strong> is a fundamental concept in
iRODS that organizes data objects into a hierarchical structure. In
iRODS, collections are similar to subdirectories and are used to group
related data objects together. These collections do not reference the
physical storage path (i.e., where the data is stored in the file
system), allowing data objects within the same collection to be stored
in different physical locations. Additionally, a data object in iRODS
can have multiple replicas, which are exact copies of the file stored in
different locations.</p>
<p>IRODs server has many features that are beyond the scope of this
material.</p>
<div class="section level4">
<h4 id="davrods">Davrods<a class="anchor" aria-label="anchor" href="#davrods"></a>
</h4>
<p>Davrods provides access to iRODS servers using the WebDAV protocol, a
similar protocol to the one used by the CardioCloud. This feature
enables users to link external tools, such as Jupyter notebooks,
Integrated Genome Viewer (IGV), and Galaxy tools.</p>
</div>
<div class="section level3">
<h3 id="sodar-naming-and-concepts">SODAR naming and concepts<a class="anchor" aria-label="anchor" href="#sodar-naming-and-concepts"></a>
</h3>
<figure><img src="../fig/clipboard-3896476926.png" alt="Figure 3. SODAR project overview page." class="figure mx-auto d-block"><div class="figcaption">
<strong>Figure 3.</strong> SODAR project
overview page.</div>
</figure><div class="section level4">
<h4 id="web-ui">Web-UI<a class="anchor" aria-label="anchor" href="#web-ui"></a>
</h4>
<p>SODAR web-UI is the main interface for users to interact with the
system.</p>
<ul>
<li><p><strong>Navigation Sidebar</strong>: On the left side, there’s a
vertical navigation bar with various options such as “Project Overview”,
and once a project is selected, “Sample Sheets”, “Landing Zones”,
“Timeline,” and “Members.”</p></li>
<li><p><strong>Navigation Topbar:</strong> A <em>search</em> bar, enable
project or metadata search. <em>Help</em>, a link to the SODAR
documentation. <em>User menu</em> drop-down so users access
configurations and general information.</p></li>
<li><p><strong>Main Panel</strong>: The bulk of the UI consists of the
main panel, which changes depending on the section.</p></li>
</ul>
</div>
<div class="section level4">
<h4 id="members">Members:<a class="anchor" aria-label="anchor" href="#members"></a>
</h4>
<p>Members is a core feature for each project. This section provides an
overview of user roles within a project, determining their level of
access and capabilities. Users can be assigned various roles like Owner,
Delegate, Contributor, Guest, and Finder, each with specific rights,
from full project management to read-only access. Roles can be inherited
from parent categories and can be adjusted to the needs of the current
project. Owners and Delegates have the ability to add, update, or remove
members and roles. Members are added through a user search function, and
changes in membership are recorded in the project’s timeline. For users
not registered in SODAR, invitations can be sent via email, which expire
after a set duration. The system allows for the promotion of members
with inherited roles, but not demotion. Each project must have one local
owner, and the number of contributors and guests is unrestricted.</p>
<p>Detailed information are detailed on <a href="https://sodar-server.readthedocs.io/en/latest/ui_project_members.html" class="external-link">SODAR
manual</a>.</p>
</div>
<div class="section level4">
<h4 id="landing-zones">Landing Zones:<a class="anchor" aria-label="anchor" href="#landing-zones"></a>
</h4>
<p>Landing zones in SODAR serve as temporary, user-specific areas within
iRODS for file uploads, providing full write access to the user. Files
are uploaded to these landing zones, after which SODAR performs
validation and transfers them to a permanent, read-only sample data
repository. The Landing Zones application facilitates the management of
these uploads, accessible through the project sidebar or the SODAR home
page. Should validation or upload fail, the process is reversed to allow
corrections, ensuring the integrity of data in the repository.</p>
</div>
<div class="section level4">
<h4 id="sample-sheets">Sample sheets:<a class="anchor" aria-label="anchor" href="#sample-sheets"></a>
</h4>
<p>In SODAR, sample sheets represent the metadata for study design
within research projects. The Sample Sheets application is used for
importing, browsing, and editing this metadata, and it allows users to
access related files stored in the iRODS system. Access to the Sample
Sheets app is obtained through the project sidebar or via an icon on the
SODAR home page.</p>
</div>
<div class="section level4">
<h4 id="time-line">Time line:<a class="anchor" aria-label="anchor" href="#time-line"></a>
</h4>
<p>The Timeline application in SODAR serves as a log that tracks project
activities, such as member role assignments, file transfers from landing
zones, updates to sample sheets or errors. Each recorded event on the
timeline includes a timestamp, the type and origin of the event, the
user who initiated it, and a detailed description with links to related
objects and applications.</p>
</div>
<div class="section level4">
<h4 id="rest-api">REST-API<a class="anchor" aria-label="anchor" href="#rest-api"></a>
</h4>
<p>The SODAR server contains REST-API that allow the automation of
various process by server admins.</p>
</div>
</div>
<div class="section level3">
<h3 id="sodar-data-workflow-step-by-step-usage">SODAR data workflow step-by-step usage<a class="anchor" aria-label="anchor" href="#sodar-data-workflow-step-by-step-usage"></a>
</h3>
<p><img src="../fig/clipboard-3042118772.png" width="847" class="figure"><strong>Figure
4.</strong> SODAR data workflow.</p>
<ol style="list-style-type: decimal">
<li><p>Login into SODAR. Then create a project <a href="https://sodar.crc1550.com/project/project/create" class="external-link uri">https://sodar.crc1550.com/project/project/create</a>.</p></li>
<li><p>Create a subproject <a href="https://sodar.crc1550.com/project/project/create/f2688cf8-67f2-450d-ae69-97d5fda66faf" class="external-link uri">https://sodar.crc1550.com/project/project/create/</a>&lt;project_uid&gt;.</p></li>
<li><p>Create a sample sheet: Sample sheets &gt; Sheet operations &gt;
Create from template and <code>Select ISA-Tab Template</code>.</p></li>
<li><p>Activate the irods server on Sample Sheet section.</p></li>
<li>
<p>Files will be uploaded by the data management team. These are the
options:</p>
<ol style="list-style-type: decimal">
<li><p>Use &lt;<a href="https://github.com/bihealth/cubi-tk" class="external-link">cubi-tk</a>
<a href="https://github.com/bihealth/cubi-tk/blob/0f7f5b2d46d0f955eb70faf8741cc44033fe63ce/docs_manual/man_sodar_ingest.rst#L4" class="external-link">sodar
ingest</a>&gt; cubi-tk sodar ingest –sodar-url <url> –sodar-api-token
<api_token><file><landing_zone_uid></landing_zone_uid></file></api_token></url></p></li>
<li><p>Use iRODs &lt;<a href="https://docs.irods.org/master/icommands/user/#iput" class="external-link">iput</a>&gt;</p></li>
</ol>
</li>
<li><p>Call SODAR validation to check whether the data upload was
successful.</p></li>
<li><p>Modify or fix any entries, if required.</p></li>
</ol>
</div>
<div class="section level3">
<h3 id="creating-and-filling-a-sample-sheet">Creating and filling a sample sheet<a class="anchor" aria-label="anchor" href="#creating-and-filling-a-sample-sheet"></a>
</h3>
<ol style="list-style-type: decimal">
<li>Pick <code>Sample Sheets</code> on left column.</li>
<li>On Sheet operation, select create from template</li>
</ol>
<figure><img src="../fig/clipboard-1391538419.png" alt="Sheet operation dropdown" class="figure mx-auto d-block"><div class="figcaption">Sheet operation dropdown</div>
</figure><ol start="3" style="list-style-type: decimal">
<li>Select an ISA-Tab template</li>
</ol>
<figure><img src="../fig/clipboard-2875853019.png" alt="Select ISA-tab template" class="figure mx-auto d-block"><div class="figcaption">Select ISA-tab template</div>
</figure><ol start="4" style="list-style-type: decimal">
<li>Fill the form:</li>
</ol>
<figure><img src="../fig/clipboard-934212931.png" alt="ISA-tab form" class="figure mx-auto d-block"><div class="figcaption">ISA-tab form</div>
</figure><div id="note" class="callout">
<div class="callout-square">
<i class="callout-icon" data-feather="bell"></i>
</div>
<div id="note" class="callout-inner">
<h3 class="callout-title">Note</h3>
<div class="callout-content">
<p>The current templates are not fully optimized for our use-case. Fell
free to reach out to the data management team if you have question,
concerns or suggestions.</p>
</div>
</div>
</div>
<div class="section level4">
<h4 id="standard-operating-procedure-for-data-and-metadata-registry-with-sodar-service">Standard Operating Procedure for Data and Metadata Registry with
SODAR Service<a class="anchor" aria-label="anchor" href="#standard-operating-procedure-for-data-and-metadata-registry-with-sodar-service"></a>
</h4>
<p>The SOP is available here: <a href="https://cloud.crc1550.com/index.php/s/nW6CGRiDHPtP5KP" class="external-link uri">https://cloud.crc1550.com/index.php/s/nW6CGRiDHPtP5KP</a></p>
</div>
</div>
<div class="section level2">
<h2 id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<ol style="list-style-type: decimal">
<li><p><a href="https://sodar-server.readthedocs.io/en/latest/data_transfer_irods.html" class="external-link">Documentation
on SODAR data transfer</a></p></li>
<li><p><a href="https://gist.github.com/tbrittoborges/5a7ee1e3b59be89ddd66aa90c80583fd" class="external-link">Current
approach used for data transfer</a></p></li>
<li><p><a href="https://www.youtube.com/watch?v=LQ8foUpjnqs" class="external-link">SODAR
overview</a></p></li>
</ol>
</div>
<div class="section level2">
<h2 id="section"></h2>
</div>
</div>
<div class="section level1">
<h1 id="faq">FAQ:<a class="anchor" aria-label="anchor" href="#faq"></a>
</h1>
<div class="section level2">
<h2 id="how-to-access-irods">How to access iRODS:<a class="anchor" aria-label="anchor" href="#how-to-access-irods"></a>
</h2>
<p>iRODS directories can be accessed via the project pages <a href="https://sodar-server.readthedocs.io/en/latest/data_transfer_irods.html" class="external-link uri">https://sodar-server.readthedocs.io/en/latest/data_transfer_irods.html</a>
but users don’t direct acces to the iRODS server</p>
<div class="section level3">
<h3 id="why-cubi-tk-sodar-ingest-fasq-is-not-working">Why &lt;cubi-tk sodar ingest-fasq&gt; is not working?<a class="anchor" aria-label="anchor" href="#why-cubi-tk-sodar-ingest-fasq-is-not-working"></a>
</h3>
<p>The cubi-tk is a command line tool that allows to interact with SODAR
server. The command <code>cubi-tk sodar ingest-fasq</code> because of
configuration error. Use &lt;cubi-tk sodar ingest&gt;. See
implementation details on <a href="https://github.com/bihealth/cubi-tk/blob/0f7f5b2d46d0f955eb70faf8741cc44033fe63ce/cubi_tk/sodar/ingest.py" class="external-link">cubi-tk</a>.</p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul>
<li>SODAR is a user friendly data management solution that adheres to
the FAIR framework.</li>
</ul>
</div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</div>
</div>
</div></section><section id="aio-sra"><p>Content from <a href="sra.html">SRA</a></p>
<hr>
<p>Last updated on 2024-10-08 |

        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/episodes/sra.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 12 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>What is SRA and how to use it for data deposition?</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Get to know the SRA database</li>
<li>Understand the requirements for data deposition</li>
<li>Learn required optional field used for a RNA-sequencing
experiment</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section><h2 class="section-heading" id="introduction">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h2>
<hr class="half-width">
<p>The Sequence Read Archive (SRA) is a public repository that contains
high-throughput sequencing data. It is a part of the International
Nucleotide Sequence Database Collaboration (INSDC), which includes
several major institutes: the National Center for Biotechnology
Information (NCBI) in the USA, the European Bioinformatics Institute
(EBI) in Europe, and the DNA Data Bank of Japan (DDBJ) in Japan. The SRA
includes data from all life forms, as well as metagenomics and
ecological studies. The SRA stores raw sequencing data and sometimes the
processed alignment data. Researchers often use the SRA to deposit their
data, which is typically a requirement for the publication of research
papers. It aims to establish a central repository for next-generation
sequencing data, linking to resources that reference or utilize this
data. The repository allows in tracking project metadata for studies and
experiments. There’s a focus on facilitating flexible submission and
retrieval of ancillary data, providing normalized data structures.
Additionally, the objectives include decoupling the submission process
from content and laying the groundwork for interactive user submissions
and data retrieval.</p>
</section><section><h2 class="section-heading" id="sra-concepts">SRA concepts:<a class="anchor" aria-label="anchor" href="#sra-concepts"></a>
</h2>
<hr class="half-width">
<p>SRA metadata concept separates experimental data from its metadata,
organizing the latter into a structured hierarchy:</p>
<ul>
<li><p><strong>Study</strong>: Represents a collection of experiments
aimed at achieving a common goal, serving as an overarching project that
encapsulates the purpose and scope of the research conducted. A study
provides the context and rationale for the experiments.</p></li>
<li><p><strong>Experiment</strong>: Refers to a specific set of
laboratory procedures applied to input material, designed to achieve an
anticipated result. Each experiment is a component of a study,
contributing to its overall objective. It can also be interpreted as a
series of experimental protocols or assays conducted to test a
hypothesis or gather data.</p></li>
<li><p><strong>Sample</strong>: The focus of an experiment, which can be
a single sample or multiple samples grouped together. The results of an
experiment are articulated based on these samples, detailing the
outcomes for individual samples or their collective group as defined by
the experimental setup. A sample is not equivalent to an individual, as
one individual can provide multiple samples (e.g., right and left
ventricle tissue samples).</p></li>
<li><p><strong>Run</strong>: Denotes the actual outcomes or results of
the experiment. These runs encompass the data collected from a sample or
a group of samples linked to a specific experiment. Essentially, a run
is the execution of the sequencing or analytical process that generates
data.</p></li>
<li><p><strong>Submission</strong>: A submission encompasses a bundle of
metadata and data objects, coupled with instructions on how the
submission should be handled. This package facilitates the organized and
controlled entry of both experimental data and its corresponding
metadata into the repository, ensuring that the data can be accurately
classified, accessed, and utilized.</p></li>
</ul></section><section><h2 class="section-heading" id="accessing-the-sra">Accessing the SRA<a class="anchor" aria-label="anchor" href="#accessing-the-sra"></a>
</h2>
<hr class="half-width">
<p>The SRA database can be accessed through the <a href="https://www.ncbi.nlm.nih.gov/sra" class="external-link">NCBI website</a>.</p>
<p>Although discussing data access is outside the scope of this
material, it is important to note that data access is a key aspect of
the FAIR principles. This means that with the right metadata, data can
be reused, generating more follow-up studies and citations.</p>
<p>Significant to note that not all data on the SRA is public, some data
is private and only accessible to authorized users. Data can also be
embargoed for a certain period of time. Recently, SRA started to use
placeholder as a way to keep track of data produced by large cohort and
require controlled access.</p>
</section><section><h2 class="section-heading" id="data-deposition">Data deposition<a class="anchor" aria-label="anchor" href="#data-deposition"></a>
</h2>
<hr class="half-width">
<p>There are multiple ways to deposit data on the SRA, the most common
is through the submission portal. This approach is detailed <a href="https://www.ncbi.nlm.nih.gov/sra/docs/submitportal/" class="external-link">here</a>. At
the CRC, we plan to streamline this process using a programmatic
approach for data submission, as detailed in this <a href="https://cloud.crc1550.com/index.php/s/NYkwKpMbHP3GTBn" class="external-link">SOP</a>. We
plan to automatize this process in the future.</p>
</section><section><h2 class="section-heading" id="metadata-for-a-transcriptomics-experiment">Metadata for a transcriptomics experiment<a class="anchor" aria-label="anchor" href="#metadata-for-a-transcriptomics-experiment"></a>
</h2>
<hr class="half-width">
<p>As you can realize, there are differences between the ISA framework
and SRA. To enable the unambiguous interpretation and adhere to the FAIR
framework, we take some pragmatic rules to define and create the
Investigation, Study, and Assay files. The following table shows the
required and recommended fields for an RNA-sequencing experiment, and
can be easily adapted to other types of experiments.</p>
<div class="section level3">
<h3 id="investigation-metadata">Investigation metadata<a class="anchor" aria-label="anchor" href="#investigation-metadata"></a>
</h3>
<table class="table">
<colgroup>
<col width="21%">
<col width="21%">
<col width="28%">
<col width="27%">
</colgroup>
<thead><tr class="header">
<th>Metadata Field</th>
<th>Required?</th>
<th>Definition</th>
<th>Comment</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>Identifier</td>
<td>required</td>
<td>Unique identifier for the investigation</td>
<td></td>
</tr>
<tr class="even">
<td>Title</td>
<td>required</td>
<td>Title of the investigation</td>
<td></td>
</tr>
<tr class="odd">
<td>Description</td>
<td>required</td>
<td>Brief description of the investigation</td>
<td></td>
</tr>
<tr class="even">
<td>Submission Date</td>
<td>required</td>
<td>Date the investigation was submitted</td>
<td>Standardized date format</td>
</tr>
<tr class="odd">
<td>Public Release Date</td>
<td>required</td>
<td>Date the investigation was publicly released</td>
<td>Standardized date format</td>
</tr>
<tr class="even">
<td>Contacts</td>
<td>required</td>
<td>List of contacts associated with the investigation</td>
<td>Names, affiliations, roles, and contact details</td>
</tr>
<tr class="odd">
<td>Study Design Types</td>
<td>recommended</td>
<td>Types of study designs</td>
<td></td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="sample-metadata1">Sample metadata<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>
<a class="anchor" aria-label="anchor" href="#sample-metadata1"></a>
</h3>
<table class="table">
<colgroup>
<col width="20%">
<col width="20%">
<col width="20%">
<col width="37%">
</colgroup>
<thead><tr class="header">
<th>Metadata Field</th>
<th>Required?</th>
<th>Definition</th>
<th>Comment</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>unique ID</td>
<td>required</td>
<td>Identifier for a sample that is at least unique within the
project</td>
<td></td>
</tr>
<tr class="even">
<td>sample type</td>
<td>required</td>
<td>The type of the collected specimen, e.g., tissue biopsy, blood draw,
or throat swab</td>
<td>Ontology field - e.g., OBI or EFO</td>
</tr>
<tr class="odd">
<td>species</td>
<td>required</td>
<td>The primary species of the specimen, preferably the taxonomic
identifier</td>
<td>Ontology field - NCBITaxonomy</td>
</tr>
<tr class="even">
<td>tissue/organism part</td>
<td>required</td>
<td>The tissue from which the sample was taken</td>
<td>Ontology field - e.g., Uberon</td>
</tr>
<tr class="odd">
<td>disease</td>
<td>required</td>
<td>Any diseases that may affect the sample</td>
<td>This may not necessarily be the same as the host’s disease, e.g.,
healthy brain tissue might be collected from a host with type II
diabetes while cirrhotic liver tissue might be collected from an
otherwise healthy individual. Ontology field - e.g., MONDO or DO</td>
</tr>
<tr class="even">
<td>sex</td>
<td>required</td>
<td>The biological/genetic sex of the sample</td>
<td>Ontology field - e.g., PATO</td>
</tr>
<tr class="odd">
<td>development stage</td>
<td>required</td>
<td>The developmental stage of the sample</td>
<td>Ontology field - e.g., Uberon or Hsadpdv; species dependent</td>
</tr>
<tr class="even">
<td>collection date</td>
<td>required</td>
<td>The date on which the sample was collected, in a standardized
format</td>
<td>Collection date in combination with other fields such as location
and disease may be sufficient to de-anonymize a sample</td>
</tr>
<tr class="odd">
<td>external accessions</td>
<td>recommended</td>
<td>Accession numbers from any external resources to which the sample
was submitted</td>
<td>e.g., Biosamples, Biostudies</td>
</tr>
<tr class="even">
<td>ancestry/ethnicity</td>
<td>recommended</td>
<td>Ancestry or ethnic group of the individual from which the sample was
collected</td>
<td>Ontology field - e.g., HANCESTRO</td>
</tr>
<tr class="odd">
<td>age</td>
<td>recommended</td>
<td>Age of the organism from which the sample was collected</td>
<td></td>
</tr>
<tr class="even">
<td>age unit</td>
<td>recommended</td>
<td>Unit of the value of the age field</td>
<td>Ontology field - e.g., UO</td>
</tr>
<tr class="odd">
<td>BMI</td>
<td>recommended</td>
<td>Body mass index of the individual from which the sample was
collected</td>
<td>Only applies to human samples</td>
</tr>
<tr class="even">
<td>treatment category</td>
<td>recommended</td>
<td>Treatments that the sample might have undergone after
collection</td>
<td>Ontology field - e.g., OBI, NCIt, or OGMS</td>
</tr>
<tr class="odd">
<td>cell type</td>
<td>recommended</td>
<td>The cell type(s) known or selected to be present in the sample</td>
<td>Ontology field - e.g., CL</td>
</tr>
<tr class="even">
<td>growth conditions</td>
<td>recommended</td>
<td>Features relating to the growth and/or maintenance of the
sample</td>
<td></td>
</tr>
<tr class="odd">
<td>genetic variation</td>
<td>recommended</td>
<td>Any relevant genetic differences from the specimen or sample to the
expected genomic information for this species, e.g., abnormal chromosome
counts, major translocations, or indels</td>
<td></td>
</tr>
<tr class="even">
<td>sample collection technique</td>
<td>recommended</td>
<td>The technique used to collect the specimen, e.g., blood draw or
surgical resection</td>
<td>Ontology field - e.g., EFO or OBI</td>
</tr>
<tr class="odd">
<td>phenotype</td>
<td>recommended</td>
<td>Any relevant (usually abnormal) phenotypes of the specimen or
sample</td>
<td>Ontology field - e.g., HP or MP; species dependent</td>
</tr>
<tr class="even">
<td>cell cycle</td>
<td>recommended</td>
<td>The cell cycle phase of the sample (for synchronized growing cells
or a single-cell sample), if known</td>
<td>Ontology field - e.g., GO</td>
</tr>
<tr class="odd">
<td>cell location</td>
<td>recommended</td>
<td>The cell location from which genetic material was collected (usually
either nucleus or mitochondria)</td>
<td>Ontology field - e.g., GO</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="assay-metadata">Assay metadata<a class="anchor" aria-label="anchor" href="#assay-metadata"></a>
</h3>
<table class="table">
<colgroup>
<col width="21%">
<col width="21%">
<col width="28%">
<col width="27%">
</colgroup>
<thead><tr class="header">
<th>Metadata Field</th>
<th>Required?</th>
<th>Definition</th>
<th>Comment</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>Identifier</td>
<td>required</td>
<td>Unique identifier for the assay</td>
<td></td>
</tr>
<tr class="even">
<td>File Name</td>
<td>required</td>
<td>Name of the file that contains the assay data</td>
<td></td>
</tr>
<tr class="odd">
<td>Measurement Type</td>
<td>required</td>
<td>Type of measurement performed in the assay</td>
<td>Ontology field - e.g. OBI</td>
</tr>
<tr class="even">
<td>Technology Type</td>
<td>required</td>
<td>Type of technology used in the assay</td>
<td>Ontology field - e.g. OBI</td>
</tr>
<tr class="odd">
<td>Technology Platform</td>
<td>recommended</td>
<td>Specific platform or instrument used in the assay</td>
<td></td>
</tr>
<tr class="even">
<td>Performer</td>
<td>required</td>
<td>Person or organization that performed the assay</td>
<td>Names, affiliations, roles, and contact details</td>
</tr>
<tr class="odd">
<td>Date</td>
<td>recommended</td>
<td>Date the assay was performed</td>
<td>Standardized date format</td>
</tr>
<tr class="even">
<td>Parameter Values</td>
<td>recommended</td>
<td>Parameters and their values used in the assay</td>
<td>Names, types, descriptions, and values</td>
</tr>
<tr class="odd">
<td>Sample Name</td>
<td>required</td>
<td>Name of the sample used in the assay</td>
<td></td>
</tr>
<tr class="even">
<td>Raw Data File</td>
<td>recommended</td>
<td>File name of the raw data generated by the assay</td>
<td></td>
</tr>
<tr class="odd">
<td>Processed Data File</td>
<td>recommended</td>
<td>File name of the processed data generated by the assay</td>
<td></td>
</tr>
<tr class="even">
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul>
<li>The SRA serves is public repository for high-throughput sequencing
data, supporting a wide range of genomic research by providing access to
raw sequencing data, alignments and feature counts.</li>
<li>Metadata Organization needs to be organized. Although ISA-files
don’t directly map to SRA metadata model, there is a clear separation
between required and optional items. There is also some flexibility to
adapt the SODAR system to a given experiment.</li>
<li>While access to the SRA database is facilitated through the NCBI
website, the database also highlights the principles of controlled
access and privacy for certain data sets.</li>
</ul>
</div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</div>
</section><div class="footnotes footnotes-end-of-document">
<hr>
<ol>
<li id="fn1"><p>Based on <a href="https://faircookbook.elixir-europe.org/content/recipes/interoperability/transcriptomics-metadata.html#suggested-metadata-fields" class="external-link">The
faircookbook</a><a href="#fnref1" class="footnote-back">↩︎</a></p></li>
</ol>
</div></section>
</div>
    </main>
</div>
<!-- END  : inst/pkgdown/templates/content-extra.html -->

      </div>
<!--/div.row-->
      		<footer class="row footer mx-md-3"><hr>
<div class="col-md-6">
        <p>This lesson is subject to the <a href="CODE_OF_CONDUCT.html">Code of Conduct</a></p>
        <p>

        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/README.md" class="external-link">Edit on GitHub</a>

	
        | <a href="https://github.com/tbrittoborges/omics_metadata/blob/main/CONTRIBUTING.md" class="external-link">Contributing</a>
        | <a href="https://github.com/tbrittoborges/omics_metadata/" class="external-link">Source</a></p>
				<p><a href="https://github.com/tbrittoborges/omics_metadata/blob/main/CITATION" class="external-link">Cite</a> | <a href="mailto:thiago.brittoborges@uni-heidelberg.de">Contact</a> | <a href="https://carpentries.org/about/" class="external-link">About</a></p>
			</div>
			<div class="col-md-6">

        <p>Materials licensed under <a href="LICENSE.html">CC-BY 4.0</a> by the authors</p>

        <p>Template licensed under <a href="https://creativecommons.org/licenses/by-sa/4.0/" class="external-link">CC-BY 4.0</a> by <a href="https://carpentries.org/" class="external-link">The Carpentries</a></p>
        <p>Built with <a href="https://github.com/carpentries/sandpaper/tree/0.16.7" class="external-link">sandpaper (0.16.7)</a>, <a href="https://github.com/carpentries/pegboard/tree/0.7.6" class="external-link">pegboard (0.7.6)</a>, and <a href="https://github.com/carpentries/varnish/tree/1.0.4" class="external-link">varnish (1.0.4)</a></p>
			</div>
		</footer>
</div> <!-- / div.container -->
	<div id="to-top">
		<a href="#top">
      <i class="search-icon" data-feather="arrow-up" role="img" aria-label="Back To Top"></i><br><!-- <span class="d-none d-sm-none d-md-none d-lg-none d-xl-block">Back</span> To Top --><span class="d-none d-sm-none d-md-none d-lg-none d-xl-block">Back</span> To Top
		</a>
	</div>
  <script type="application/ld+json">
    {
  "@context": "https://schema.org",
  "@type": "TrainingMaterial",
  "@id": "https://tbrittoborges.github.io/omics_metadata/instructor/aio.html",
  "inLanguage": "en",
  "dct:conformsTo": "https://bioschemas.org/profiles/TrainingMaterial/1.0-RELEASE",
  "description": "A Carpentries Lesson teaching foundational data and coding skills to researchers worldwide",
  "keywords": "data, metadata, bioinformatics, data management",
  "name": "All in One View",
  "creativeWorkStatus": "active",
  "url": "https://tbrittoborges.github.io/omics_metadata/instructor/aio.html",
  "identifier": "https://tbrittoborges.github.io/omics_metadata/instructor/aio.html",
  "dateCreated": "2024-02-14",
  "dateModified": "2024-10-08",
  "datePublished": "2024-10-08"
}

  </script><script>
		feather.replace();
	</script>
</body>
</html><!-- END:   inst/pkgdown/templates/layout.html-->

