<!DOCTYPE html>
<!-- START: inst/pkgdown/templates/layout.html --><!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<title>CRC1550 data management training material: All in One View</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" type="text/css" href="../assets/styles.css">
<script src="../assets/scripts.js" type="text/javascript"></script><!-- mathjax --><script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      config: ["MMLorHTML.js"],
      jax: ["input/TeX","input/MathML","output/HTML-CSS","output/NativeMML", "output/PreviewHTML"],
      extensions: ["tex2jax.js","mml2jax.js","MathMenu.js","MathZoom.js", "fast-preview.js", "AssistiveMML.js", "a11y/accessibility-menu.js"],
      TeX: {
        extensions: ["AMSmath.js","AMSsymbols.js","noErrors.js","noUndefined.js"]
      },
      tex2jax: {
        inlineMath: [['\\(', '\\)']],
        displayMath: [ ['$$','$$'], ['\\[', '\\]'] ],
        processEscapes: true
      }
    });
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><!-- Responsive Favicon for The Carpentries --><link rel="apple-touch-icon" sizes="180x180" href="../apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="manifest" href="../site.webmanifest">
<link rel="mask-icon" href="../safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#da532c">
<meta name="theme-color" content="#ffffff">
</head>
<body>
    <header id="top" class="navbar navbar-expand-md navbar-light bg-white top-nav incubator"><a class="visually-hidden-focusable skip-link" href="#main-content">Skip to main content</a>
  <div class="container-fluid top-nav-container">
    <div class="col-md-6">
      <div class="large-logo">
        <img alt="Carpentries Incubator" src="../assets/images/incubator-logo.svg"><abbr class="badge badge-light" title="This lesson is in the pre-alpha phase, which means that it is in early development, but has not yet been taught." style="background-color: #FF4955; border-radius: 5px">
          <a href="https://cdh.carpentries.org/the-lesson-life-cycle.html#early-development-pre-alpha-through-alpha" class="external-link alert-link" style="color: #000">
            <i aria-hidden="true" class="icon" data-feather="alert-octagon" style="border-radius: 5px"></i>
            Pre-Alpha
          </a>
          <span class="visually-hidden">This lesson is in the pre-alpha phase, which means that it is in early development, but has not yet been taught.</span>
        </abbr>
        
      </div>
    </div>
    <div class="selector-container">
      
      
      <div class="dropdown">
        <button class="btn btn-secondary dropdown-toggle bordered-button" type="button" id="dropdownMenu1" data-bs-toggle="dropdown" aria-expanded="false">
          <i aria-hidden="true" class="icon" data-feather="eye"></i> Instructor View <i data-feather="chevron-down"></i>
        </button>
        <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
<li><button class="dropdown-item" type="button" onclick="window.location.href='../aio.html';">Learner View</button></li>
        </ul>
</div>
    </div>
  </div>
  <hr></header><nav class="navbar navbar-expand-xl navbar-light bg-white bottom-nav incubator" aria-label="Main Navigation"><div class="container-fluid nav-container">
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle Navigation">
      <span class="navbar-toggler-icon"></span>
      <span class="menu-title">Menu</span>
    </button>
    <div class="nav-logo">
      <img class="small-logo" alt="Carpentries Incubator" src="../assets/images/incubator-logo-sm.svg">
</div>
    <div class="lesson-title-md">
      CRC1550 data management training material
    </div>
    <div class="search-icon-sm">
      <!-- TODO: do not show until we have search
        <i role="img" aria-label="search button" data-feather="search"></i>
      -->
    </div>
    <div class="desktop-nav">
      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
<li class="nav-item">
          <span class="lesson-title">
            CRC1550 data management training material
          </span>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/key-points.html">Key Points</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/instructor-notes.html">Instructor Notes</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/images.html">Extract All Images</a>
        </li>
        <li class="nav-item dropdown">
          <button class="nav-link dropdown-toggle" id="navbarDropdown" data-bs-toggle="dropdown" aria-expanded="false">
            More <i data-feather="chevron-down"></i>
          </button>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
<hr>
<li><a class="dropdown-item" href="reference.html">Reference</a></li>
          </ul>
</li>
      </ul>
</div>
    <form class="d-flex col-md-2 search-form">
      <fieldset disabled>
<input class="form-control me-2 searchbox" type="search" placeholder="Search" aria-label="Search"><button class="btn btn-outline-success tablet-search-button" type="submit">
          <i class="search-icon" data-feather="search" role="img" aria-label="search button"></i>
        </button>
      </fieldset>
</form>
  </div>
<!--/div.container-fluid -->
</nav><div class="col-md-12 mobile-title">
  CRC1550 data management training material
</div>

<aside class="col-md-12 lesson-progress"><div style="width: %" class="percentage">
    %
  </div>
  <div class="progress incubator">
    <div class="progress-bar incubator" role="progressbar" style="width: %" aria-valuenow="" aria-label="Lesson Progress" aria-valuemin="0" aria-valuemax="100">
    </div>
  </div>
</aside><div class="container">
      <div class="row">
        <!-- START: inst/pkgdown/templates/navbar.html -->
<div id="sidebar-col" class="col-lg-4">
  <div id="sidebar" class="sidebar">
      <nav aria-labelledby="flush-headingEleven"><button role="button" aria-label="close menu" alt="close menu" aria-expanded="true" aria-controls="sidebar" class="collapse-toggle" data-collapse="Collapse " data-episodes="Episodes ">
          <i class="search-icon" data-feather="x" role="img"></i>
        </button>
        <div class="sidebar-inner">
          <div class="row mobile-row">
            <div class="col">
              <div class="sidenav-view-selector">
                <div class="accordion accordion-flush" id="accordionFlush9">
                  <div class="accordion-item">
                    <h2 class="accordion-header" id="flush-headingNine">
                      <button class="accordion-button collapsed" id="instructor" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseNine" aria-expanded="false" aria-controls="flush-collapseNine">
                        <i id="eye" aria-hidden="true" class="icon" data-feather="eye"></i> Instructor View
                      </button>
                    </h2>
                    <div id="flush-collapseNine" class="accordion-collapse collapse" aria-labelledby="flush-headingNine" data-bs-parent="#accordionFlush2">
                      <div class="accordion-body">
                        <a href="../aio.html">Learner View</a>
                      </div>
                    </div>
                  </div>
<!--/div.accordion-item-->
                </div>
<!--/div.accordion-flush-->
              </div>
<!--div.sidenav-view-selector -->
            </div>
<!--/div.col -->
      
            <hr>
</div>
<!--/div.mobile-row -->

          <div class="accordion accordion-flush" id="accordionFlush11">
            <div class="accordion-item">

              <button id="chapters" class="accordion-button show" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseEleven" aria-expanded="false" aria-controls="flush-collapseEleven">
                <h2 class="accordion-header chapters" id="flush-headingEleven">
                  EPISODES
                </h2>
              </button>
              <div id="flush-collapseEleven" class="accordion-collapse show collapse" aria-labelledby="flush-headingEleven" data-bs-parent="#accordionFlush11">

                <div class="accordion-body">
                  <div class="accordion accordion-flush" id="accordionFlush1">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading1">
        <a href="index.html">Summary and Schedule</a>
    </div>
<!--/div.accordion-header-->
        
  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush2">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading2">
        <a href="introduction.html">1. Preamble to Data Management Plan</a>
    </div>
<!--/div.accordion-header-->
        
  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush3">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading3">
        <a href="bionfo_primer.html">2. Primer in Bioinformatics</a>
    </div>
<!--/div.accordion-header-->
        
  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush4">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading4">
        <a href="fair-principles.html">3. fair_principles</a>
    </div>
<!--/div.accordion-header-->
        
  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush5">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading5">
        <a href="sodar.html">4. SODAR: System for Omics Data Access and Retrieval.</a>
    </div>
<!--/div.accordion-header-->
        
  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush6">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading6">
        <a href="data_management_plan.html">5. Data Management Plan</a>
    </div>
<!--/div.accordion-header-->
        
  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush7">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading7">
        <a href="sra.html">6. SRA</a>
    </div>
<!--/div.accordion-header-->
        
  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

                </div>
              </div>
            </div>

            <hr class="half-width">
<div class="accordion accordion-flush resources" id="accordionFlush12">
              <div class="accordion-item">
                <h2 class="accordion-header" id="flush-headingTwelve">
                  <button class="accordion-button collapsed" id="resources" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseTwelve" aria-expanded="false" aria-controls="flush-collapseTwelve">
                    RESOURCES
                  </button>
                </h2>
                <div id="flush-collapseTwelve" class="accordion-collapse collapse" aria-labelledby="flush-headingTwelve" data-bs-parent="#accordionFlush12">
                  <div class="accordion-body">
                    <ul>
<li>
                        <a href="../instructor/key-points.html">Key Points</a>
                      </li>
                      <li>
                        <a href="../instructor/instructor-notes.html">Instructor Notes</a>
                      </li>
                      <li>
                        <a href="../instructor/images.html">Extract All Images</a>
                      </li>
                      <hr>
<li><a class="dropdown-item" href="reference.html">Reference</a></li>
                    </ul>
</div>
                </div>
              </div>
            </div>
            <hr class="half-width resources">
<a href="../instructor/aio.html">See all in one page</a>
            

            <hr class="d-none d-sm-block d-md-none">
<div class="d-grid gap-1">
            
            </div>
          </div>
<!-- /div.accordion -->
        </div>
<!-- /div.sidebar-inner -->
      </nav>
</div>
<!-- /div.sidebar -->
  </div>
<!-- /div.sidebar-col -->
<!-- END:   inst/pkgdown/templates/navbar.html-->

        <!-- START: inst/pkgdown/templates/content-extra.html -->
  <div class="col-xl-8 col-lg-12 primary-content">
    <main id="main-content" class="main-content"><div class="container lesson-content">
        
        
<section id="aio-introduction"><p>Content from <a href="introduction.html">Preamble to Data Management Plan</a></p>
<hr>
<p>Last updated on 2024-03-22 |
        
        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/episodes/introduction.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 0 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>None</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>What is Metadata?</li>
<li>What is the differentiate between data and metadata.</li>
<li>Data management and metadata standardization.</li>
<li>Understanding its importance in OMICs experiments.</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section id="introduction"><h2 class="section-heading">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h2>
<hr class="half-width">
<p>In this lesson, we will learn about metadata and its importance in
data analysis. We will learn about the different types of metadata and
how it is used to provide context and meaning to data. We will also
learn about the importance of metadata standardization and how it can
help to improve data management and data analysis. Finally, we will
learn about the importance of metadata in OMICs experiments and how it
can help to improve the reproducibility and reliability of research
results.</p>
<div class="section level3">
<h3 id="what-is-metadata">What is Metadata?<a class="anchor" aria-label="anchor" href="#what-is-metadata"></a>
</h3>
<p>Metadata is data that provides information about other data. It is an
essential component of any data analysis pipeline. It is used to
describe the content, quality, condition, and other characteristics of
data. Metadata is used to provide context and meaning to data. It is
used to help users understand the data and to help them make informed
decisions about how to use it.</p>
</div>
<div class="section level3">
<h3 id="data-vs--metadata">Data vs. Metadata<a class="anchor" aria-label="anchor" href="#data-vs--metadata"></a>
</h3>
<p>Data is the raw information that is collected and analyzed in a
research study. It is the information that is used to answer research
questions and to make decisions. Metadata is the information that
describes the data. It is the information that provides context and
meaning to the data. It is used to help users understand the data and to
help them make informed decisions about how to use it.</p>
<p>An every day analogy is digital photography. Every time you take a
photo with a digital camera, a range of “behind-the-scenes” information
is saved along with the image data. This is known as metadata, and it
can include details of the camera, lens and shooting settings used, plus
optional information about the photographer, location and more.</p>
</div>
<div class="section level3">
<h3 id="data-management-and-metadata-standardization">Data Management and Metadata Standardization<a class="anchor" aria-label="anchor" href="#data-management-and-metadata-standardization"></a>
</h3>
<p>Metadata standardization is the process of defining and implementing
a set of rules and guidelines for the creation, management, and use of
metadata. It is used to ensure that metadata is reliable. It is used to
help users find, access, and use data more effectively.</p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points<a class="anchor" aria-label="anchor" href="#keypoints1"></a>
</h3>
<div class="callout-content">
<ul>
<li>Data and metadata are important components of any data analysis
pipeline.</li>
<li>Metadata standardization is critical for reliable results.</li>
<li>Data management plan and metadata enables data FAIRification.</li>
</ul>
</div>
</div>
</div>
<!-- 
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use. 
 -->
</div>
</section></section><section id="aio-bionfo_primer"><p>Content from <a href="bionfo_primer.html">Primer in Bioinformatics</a></p>
<hr>
<p>Last updated on 2024-03-22 |
        
        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/episodes/bionfo_primer.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 0 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>None.</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Introduce the idea of OMICs and explain why they are important in
bioinformatics.</li>
<li>Familiarize with commonly used file formats</li>
<li>Understanding the multilayered nature of metadata from OMICs
experiments.</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<p><strong>Introduction</strong>: OMICs technologies provide unbiased
measurements of molecular entities in biological samples. These
measurements allow for a comprehensive exploration of functions and
association of such entities. The extensive utilization of OMICs
techniques resulted in the establishment of system biology, a branch of
biology that aims to comprehend biological systems in their
entirety.</p>
<p><strong>OMICs</strong> is a collective term for a group of
technologies that allow for the comprehensive analysis of biological
molecules. These technologies include genomics, transcriptomics,
proteomics, metabolomics, and lipidomics. Each of these technologies
provides a unique perspective on the molecular composition of a
biological sample, and together they allow for a comprehensive
exploration of the functions and associations of biological
molecules.</p>
<section id="general-definition"><h2 class="section-heading">General definition<a class="anchor" aria-label="anchor" href="#general-definition"></a>
</h2>
<hr class="half-width">
<ul>
<li>
<strong>Genomics</strong>: Study of an organism’s entire genome,
exome or selected regions, offering insights into genetic variations.
When associated with disease phenotype or clinical data can be
implicated on health and disease.</li>
<li>
<strong>Transcriptomics</strong>: Focuses on the transcriptome, the
complete set of RNA transcripts produced by the genome, under specific
circumstances or in a specific cell.</li>
<li>
<strong>Proteomics</strong>: Examines the proteome, the entire set
of proteins produced or modified by an organism, revealing functions and
pathways.</li>
<li>
<strong>Metabolomics</strong>: Involves the study of metabolites,
providing a snapshot of the physiological condition of a cell or
organism.</li>
<li>
<strong>Lipidomics</strong>: A complete lipid profile within a cell,
tissue, or organism, allowing the study of their roles in cellular
processes.</li>
</ul></section><section id="file-formats"><h2 class="section-heading">File formats<a class="anchor" aria-label="anchor" href="#file-formats"></a>
</h2>
<hr class="half-width">
<p>There are a multitude of file formats used in bioinformatics, each
with its own specific use case. Some of the most commonly used file
formats include:</p>
<p><strong>FASTQ format</strong> is a text-based format for storing both
an oligonucleotide sequence and its corresponding quality scores.
Obtained after base calling.</p>
<p><strong>SAM/BAM/CRAM files</strong>: These are used to store sequence
alignment data, and are used to represent the alignment of reads to a
reference genome. SAM is a text-based format, while BAM is a binary
version of the same format. CRAM is a compressed version of the BAM
format.</p>
<p><strong>PRIDE XML, mzIdentML, mzTab, and mzML</strong>: These file
formats are used to store, process, and visualize mass
spectrometry-based proteomics data deposited in the PRIDE Archive. PRIDE
XML is the internal data format and submission format of PRIDE, while
mzIdentML and mzTab are standard file formats for reporting
peptide/protein identification and quantification results. Additionally,
mzML is used to store the raw output files from mass spectrometers. Note
that there are many different file formats for raw proteomics datasets,
depending on the equipment vendor.</p>
<p><strong>Raw sequencing formats:</strong> high-throughput machine such
as Illumina, Nanopore, produce files before base calling. These files
are in the form of binary files, and are not human-readable and depend
on vendor-specific methods for signal processing.</p>
<div id="accordionHint1" class="accordion challenge-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button hint-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseHint1" aria-expanded="false" aria-controls="collapseHint1">
  <h4 class="accordion-header" id="headingHint1">Give me a hint</h4>
</button>
<div id="collapseHint1" class="accordion-collapse collapse" data-bs-parent="#accordionHint1" aria-labelledby="headingHint1">
<div class="accordion-body">
<p>For bioinformatics workflow, the FASTQ format is often the initial
step. This file format stores “reads”, or nucleotide sequences, along
with their quality scores. Each sequence in a FASTQ file is represented
by four lines: a header with a ‘@’ symbol followed by an identifier and
optional description, the nucleotide sequence itself, a separator line
starting with a ‘+’, and the quality scores encoded as ASCII characters.
This structure allows for efficient storage and analysis of sequencing
data, essential for tasks like genome assembly and analysis. For more
detailed information, please visit the Wikipedia page on <a href="https://en.wikipedia.org/wiki/FASTQ_format" class="external-link">FASTQ format</a>.</p>
</div>
</div>
</div>
</div>
<p>It is out of the scope of this lesson to detail all file formats used
in bioinformatics, but it is important to understand that the different
files a specific purpose, can be vendor-specific and that the choice of
file format can have a significant impact on the downstream analysis of
the data. For example, use of legacy vendor-specific format can limit
the use of the data in modern bioinformatics pipelines, as proprietary
programs may disappear. In general, it’s good practice to use
open-source file formats.</p>
</section><section id="challenges-in-omics-data"><h2 class="section-heading">Challenges in OMICs data:<a class="anchor" aria-label="anchor" href="#challenges-in-omics-data"></a>
</h2>
<hr class="half-width">
<p>There are many challenges involved in the use of OMICs datasets.</p>
<ul>
<li>
<strong>Data storage</strong>: OMICs datasets can be large, and
require robust data storage and backup solutions. This is especially
true for raw sequencing data, which can be many gigabytes in size. We
expect the size of OMICs datasets to predict by the CRC to be in the
order of 20 terabytes per year.</li>
<li>
<strong>Data analysis</strong>: OMICs datasets are complex, and
require specialized methods for analysis.</li>
<li>
<strong>Standardization</strong>: As there are many different file
formats, there are many data standards and protocols used in OMICs
research. This lack of standardization make it difficult to compare data
between different labs.</li>
<li>
<strong>Ethical considerations</strong>: Handling of personal OMICs
data requires careful consideration of ethical and legal issues, such as
patient privacy and data sharing.</li>
<li>
<strong>Data sharing</strong>: OMICs datasets are required to be
shared upon publication.</li>
</ul>
<p>All the challenges can be mitigated through metadata use. Metadata is
data that provides information about other data, or in other words, a
detailed description of the data’s attribute. For example, for the data
sharing challenge mentioned above, metadata can be used to find and
filter datasets. This use-case is also interesting from the method
development standpoint, as new methods can be used to analyze old data,
but only if the data was properly described in terms of provenance and
metadata. Metadata is essential for understanding the context of the
data, and is used to help find and interpret the data. In the context of
OMICs data, metadata can include information about the sample donor,
sample, the experimental conditions, the data collection methods,
protocols and the data analysis methods. Metadata is essential for
reproducibility, as it allows others to understand and replicate the
results of an experiment.</p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points<a class="anchor" aria-label="anchor" href="#keypoints1"></a>
</h3>
<div class="callout-content">
<ul>
<li>OMICs technologies are crucial for a holistic understanding of
biological systems.</li>
<li>Common file formats in bioinformatics serve specific purposes,
facilitating data analysis and sharing.</li>
<li>Metadata plays a critical role in contextualizing and modeling OMICs
data.</li>
</ul>
</div>
</div>
</div>
<!-- 
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use. 
 -->
</section></section><section id="aio-fair-principles"><p>Content from <a href="fair-principles.html">fair_principles</a></p>
<hr>
<p>Last updated on 2024-03-22 |
        
        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/episodes/fair-principles.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 12 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>None</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Explain how metadata plays a pivotal role in making data FAIR</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<ul>
<li>
<strong>Findable</strong>: Metadata helps in uniquely identifying
data for easy discovery.</li>
<li>
<strong>Accessible</strong>: Metadata includes information on how
data can be accessed.</li>
<li>
<strong>Interoperable</strong>: Standardized metadata ensures that
data can be integrated with other datasets.</li>
<li>
<strong>Reusable</strong>: Metadata provides the information
necessary for data to be reused effectively, including the conditions
under which it can be reused.</li>
</ul>
<p>This is a lesson created via The Carpentries Workbench. It is written
in <a href="https://pandoc.org/MANUAL.html" class="external-link">Pandoc-flavored Markdown</a>
for static files (with extension <code>.md</code>) and <a href="https://rmarkdown.rstudio.com/" class="external-link">R Markdown</a> for dynamic files
that can render code into output (with extension <code>.Rmd</code>).
Please refer to the <a href="https://carpentries.github.io/sandpaper-docs/" class="external-link">Introduction to The
Carpentries Workbench</a> for full documentation.</p>
<p>What you need to know is that there are three sections required for a
valid Carpentries lesson template:</p>
<ol style="list-style-type: decimal">
<li>
<code>questions</code> are displayed at the beginning of the episode
to prime the learner for the content.</li>
<li>
<code>objectives</code> are the learning objectives for an episode
displayed with the questions.</li>
<li>
<code>keypoints</code> are displayed at the end of the episode to
reinforce the objectives.</li>
</ol>
<div id="accordionInstructor1" class="accordion instructor-note accordion-flush">
<div class="accordion-item">
<button class="accordion-button instructor-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseInstructor1" aria-expanded="false" aria-controls="collapseInstructor1">
  <h3 class="accordion-header" id="headingInstructor1">
<div class="note-square"><i aria-hidden="true" class="callout-icon" data-feather="edit-2"></i></div>Instructor Note</h3>
</button>
<div id="collapseInstructor1" class="accordion-collapse collapse" data-bs-parent="#accordionInstructor1" aria-labelledby="headingInstructor1">
<div class="accordion-body">
<p>Inline instructor notes can help inform instructors of timing
challenges associated with the lessons. They appear in the “Instructor
View”</p>
</div>
</div>
</div>
</div>
<div id="challenge-1-can-you-do-it" class="callout challenge">
<div class="callout-square">
<i class="callout-icon" data-feather="zap"></i>
</div>
<div id="challenge-1-can-you-do-it" class="callout-inner">
<h3 class="callout-title">Challenge 1: Can you do it?<a class="anchor" aria-label="anchor" href="#challenge-1-can-you-do-it"></a>
</h3>
<div class="callout-content">
<p>What is the output of this command?</p>
<div class="codewrapper sourceCode" id="cb1">
<h3 class="code-label">R<i aria-hidden="true" data-feather="chevron-left"></i><i aria-hidden="true" data-feather="chevron-right"></i>
</h3>
<pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">paste</span><span class="op">(</span><span class="st">"This"</span>, <span class="st">"new"</span>, <span class="st">"lesson"</span>, <span class="st">"looks"</span>, <span class="st">"good"</span><span class="op">)</span></span></code></pre>
</div>
</div>
</div>
</div>
<div id="accordionSolution1" class="accordion challenge-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button solution-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSolution1" aria-expanded="false" aria-controls="collapseSolution1">
  <h4 class="accordion-header" id="headingSolution1">Output</h4>
</button>
<div id="collapseSolution1" class="accordion-collapse collapse" data-bs-parent="#accordionSolution1" aria-labelledby="headingSolution1">
<div class="accordion-body">
<div class="codewrapper">
<h3 class="code-label">OUTPUT<i aria-hidden="true" data-feather="chevron-left"></i><i aria-hidden="true" data-feather="chevron-right"></i>
</h3>
<pre class="output" tabindex="0"><code>[1] "This new lesson looks good"</code></pre>
</div>
</div>
</div>
</div>
</div>
<div id="challenge-2-how-do-you-nest-solutions-within-challenge-blocks" class="callout challenge">
<div class="callout-square">
<i class="callout-icon" data-feather="zap"></i>
</div>
<div id="challenge-2-how-do-you-nest-solutions-within-challenge-blocks" class="callout-inner">
<h3 class="callout-title">Challenge 2: how do you nest solutions within
challenge blocks?<a class="anchor" aria-label="anchor" href="#challenge-2-how-do-you-nest-solutions-within-challenge-blocks"></a>
</h3>
<div class="callout-content">

</div>
</div>
</div>
<div id="accordionSolution2" class="accordion challenge-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button solution-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSolution2" aria-expanded="false" aria-controls="collapseSolution2">
  <h4 class="accordion-header" id="headingSolution2">Show me the solution</h4>
</button>
<div id="collapseSolution2" class="accordion-collapse collapse" data-bs-parent="#accordionSolution2" aria-labelledby="headingSolution2">
<div class="accordion-body">
<p>You can add a line with at least three colons and a
<code>solution</code> tag.</p>
</div>
</div>
</div>
</div>
<section id="figures"><h2 class="section-heading">Figures<a class="anchor" aria-label="anchor" href="#figures"></a>
</h2>
<hr class="half-width">
<p>You can include figures generated from R Markdown:</p>
<div class="codewrapper sourceCode" id="cb3">
<h3 class="code-label">R<i aria-hidden="true" data-feather="chevron-left"></i><i aria-hidden="true" data-feather="chevron-right"></i>
</h3>
<pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">pie</span><span class="op">(</span></span>
<span>  <span class="fu">c</span><span class="op">(</span>Sky <span class="op">=</span> <span class="fl">78</span>, <span class="st">"Sunny side of pyramid"</span> <span class="op">=</span> <span class="fl">17</span>, <span class="st">"Shady side of pyramid"</span> <span class="op">=</span> <span class="fl">5</span><span class="op">)</span>, </span>
<span>  init.angle <span class="op">=</span> <span class="fl">315</span>, </span>
<span>  col <span class="op">=</span> <span class="fu">c</span><span class="op">(</span><span class="st">"deepskyblue"</span>, <span class="st">"yellow"</span>, <span class="st">"yellow3"</span><span class="op">)</span>, </span>
<span>  border <span class="op">=</span> <span class="cn">FALSE</span></span>
<span><span class="op">)</span></span></code></pre>
</div>
<figure style="text-align: center"><img src="../fig/fair-principles-rendered-pyramid-1.png" alt="pie chart illusion of a pyramid" class="figure mx-auto d-block"><figcaption>
Sun arise each and every morning
</figcaption></figure><p>Or you can use pandoc markdown for static figures with the following
syntax:</p>
<p><code>![optional caption that appears below the figure](figure url){alt='alt text for accessibility purposes'}</code></p>
<figure><img src="https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg" alt="Blue Carpentries hex person logo with no text." class="figure mx-auto d-block"><div class="figcaption">You belong in The Carpentries!</div>
</figure></section><section id="math"><h2 class="section-heading">Math<a class="anchor" aria-label="anchor" href="#math"></a>
</h2>
<hr class="half-width">
<p>One of our episodes contains <span class="math inline">\(\LaTeX\)</span> equations when describing how to
create dynamic reports with {knitr}, so we now use mathjax to describe
this:</p>
<p><code>$\alpha = \dfrac{1}{(1 - \beta)^2}$</code> becomes: <span class="math inline">\(\alpha = \dfrac{1}{(1 - \beta)^2}\)</span></p>
<p>Cool, right?</p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points<a class="anchor" aria-label="anchor" href="#keypoints1"></a>
</h3>
<div class="callout-content">
<ul>
<li>Use <code>.md</code> files for episodes when you want static
content</li>
<li>Use <code>.Rmd</code> files for episodes when you need to generate
output</li>
<li>Run <code>sandpaper::check_lesson()</code> to identify any issues
with your lesson</li>
<li>Run <code>sandpaper::build_lesson()</code> to preview your lesson
locally</li>
</ul>
</div>
</div>
</div>
<!-- 
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use. 
 -->
</section></section><section id="aio-sodar"><p>Content from <a href="sodar.html">SODAR: System for Omics Data Access and Retrieval.</a></p>
<hr>
<p>Last updated on 2024-03-22 |
        
        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/episodes/sodar.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 0 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>None</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Introduces SODAR</li>
<li>Detail on main concepts of the tool</li>
<li>How to use SODAR</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section id="overview"><h2 class="section-heading">Overview<a class="anchor" aria-label="anchor" href="#overview"></a>
</h2>
<hr class="half-width">
<p><img src="../fig/SodarLogo.svg" class="figure"><strong>S</strong>ystem for
<strong>O</strong>mics <strong>D</strong>ata <strong>A</strong>ccess and
<strong>R</strong>etrieval (<strong>SODAR</strong>) is a specialized
system designed for managing data in OMICs research projects. The
platform provides medical experts with a central location for navigating
and coordinating data, connecting to other resources and systems. Key
features of SODAR include project-based access control, modeling study
design metadata, large-scale data storage, file validation, and various
tools for data management. SODAR aims to address challenges faced by
scientists working on OMICs studies by offering a user-friendly
interface for managing multi-assay studies and organizing raw dataset,
metadata and results in a single place. Most end users will use SODAR
via its web-based GUI, while data stewards will use the REST APIs. SODAR
server runs on the Django web server, models metadata with the ISA
specification, and uses iRODS for file storage. SODAR is developed by
the <a href="https://www.cubi.bihealth.org/" class="external-link">Core Unit
Bioinformatics</a> at the <a href="https://www.bihealth.org/" class="external-link">Berlin
Institute of Health</a>. The software is written in Python 3 and freely
available under the MIT license.</p>
<div class="section level3">
<h3 id="sodar-components-and-data-workflow">SODAR components and data workflow<a class="anchor" aria-label="anchor" href="#sodar-components-and-data-workflow"></a>
</h3>
<p><img src="../fig/clipboard-1870335311.png" class="figure"> Figure 1. Overview of data
workflow with SODAR. The next sections will detail each component.</p>
<div class="section level4">
<h4 id="sodar-server">SODAR server<a class="anchor" aria-label="anchor" href="#sodar-server"></a>
</h4>
<p>The SODAR server is the main component of the system. It comprises
several modules called Django apps, which are responsible for different
tasks. Although these apps are not the utmost importance for the users,
understanding the available apps allows users to understand the full
potential of the system. The main apps are detailed in the [GitHub
repository][<a href="https://github.com/bihealth/sodar-server?tab=readme-ov-file#technical-information" class="external-link uri">https://github.com/bihealth/sodar-server?tab=readme-ov-file#technical-information</a>]
of the project and listed below:</p>
<ul>
<li>
<strong>Samplesheets</strong>: Modeling of study metadata in the
ISA-Tab format</li>
<li>
<strong>Landingzones</strong>: Management of file validation and
uploads into iRODS</li>
<li>
<strong>Irodsadmin</strong>: iRODS data administration helpers</li>
<li>
<strong>Irodsbackend</strong>: Backend app for iRODS queries and
operations</li>
<li>
<strong>Irodsinfo</strong>: Display iRODS server information and
create user configurations</li>
<li>
<strong>Ontologyaccess</strong>: Parse, store and serve ontologies
for local lookup</li>
<li>
<strong>Taskflowbackend</strong>: Run iRODS transactions with full
rollback for project and file operations</li>
</ul>
<p>The combinations of these apps allow the SODAR server results the
following webpage:</p>
<p><img src="../fig/clipboard-3240582884.png" class="figure"> Figure 2: SODAR developers
provide a demo instance at <a href="https://sodar-demo.cubi.bihealth.org/samplesheets/a1e84b38-568d-44e7-b81f-43ae81927187#/study/473c5d99-532b-4c31-a056-8e58d16a6bb7" class="external-link">this
url</a>.</p>
</div>
<div class="section level4">
<h4 id="irods-server">iRODS server<a class="anchor" aria-label="anchor" href="#irods-server"></a>
</h4>
<p>iRODS stands for Integrated Rule-Oriented Data System, is the data
management system that powers SODAR data and metadata management
under-the-hood. Most user will not need to interact with iRODS directly.
iRODS provides storage virtualization among heterogeneous file systems,
meaning it allows data to be stored in different computers or external
servers (‘cloud’) and still be systematic organized in terms of
metadata. The usual use case is for organizing unstructured data, such
OMICs experiments.</p>
<p>The iCAT server serves as a metadata catalog that supports large
facility experimental data, linking all aspects of the research chain
from proposal through to publication. It utilizes a relational database
for storing metadata in the form of “triples,” consisting of an
attribute field, a value field, and a unit field.</p>
<p>An <strong>iRODS collection</strong> is a fundamental concept in
iRODS that organizes data objects into a hierarchical structure. In
iRODS, collections are similar to subdirectories and are used to group
related data objects together. These collections do not reference the
physical storage path (i.e., where the data is stored in the file
system), allowing data objects within the same collection to be stored
in different physical locations. Additionally, a data object in iRODS
can have multiple replicas, which are exact copies of the file stored in
different locations2</p>
<p>IRODs server has many features that are beyond the scope of this
material.</p>
</div>
<div class="section level4">
<h4 id="davrods">Davrods<a class="anchor" aria-label="anchor" href="#davrods"></a>
</h4>
<p>Davrods provides access to iRODS servers using the WebDAV protocol, a
similar protocol to the one used by the CardioCloud. This feature
enables users to link external tools, such as Jupyter notebooks,
Integrated Genome Viewer (IGV), and Galaxy tools.</p>
</div>
</div>
<div class="section level3">
<h3 id="sodar-naming-and-concepts">SODAR naming and concepts<a class="anchor" aria-label="anchor" href="#sodar-naming-and-concepts"></a>
</h3>
<p><img src="../fig/clipboard-3896476926.png" class="figure"> Figure 3. SODAR project
overview page.</p>
<div class="section level4">
<h4 id="web-ui">Web-UI<a class="anchor" aria-label="anchor" href="#web-ui"></a>
</h4>
<p>SODAR web-UI is the main interface for users to interact with the
system.</p>
<ul>
<li><p><strong>Navigation Sidebar</strong>: On the left side, there’s a
vertical navigation bar with various options such as “Project Overview”,
and once a project is selected, “Sample Sheets”, “Landing Zones”,
“Timeline,” and “Members.”</p></li>
<li><p><strong>Navigation Topbar:</strong> A <em>search</em> bar, enable
project or metadata search. <em>Manual</em>,a link to the SODAR
documentation. <em>User menu</em> drop-down so users access
configurations and general information.</p></li>
<li><p><strong>Main Panel</strong>: The bulk of the UI consists of the
main panel, which changes depending on the section.</p></li>
</ul>
</div>
<div class="section level4">
<h4 id="members">Members:<a class="anchor" aria-label="anchor" href="#members"></a>
</h4>
<p>Members is a core feature for each project. This section provides an
overview of user roles within a project, determining their level of
access and capabilities. Users can be assigned various roles like Owner,
Delegate, Contributor, Guest, and Finder, each with specific rights,
from full project management to read-only access. Roles can be inherited
from parent categories and can be adjusted to the needs of the current
project. Owners and Delegates have the ability to add, update, or remove
members and roles. Members are added through a user search function, and
changes in membership are recorded in the project’s timeline. For users
not registered in SODAR, invitations can be sent via email, which expire
after a set duration. The system allows for the promotion of members
with inherited roles, but not demotion. Each project must have one local
owner, and the number of contributors and guests is unrestricted.</p>
<p>Detailed information are detailed on <a href="https://sodar-server.readthedocs.io/en/latest/ui_project_members.html" class="external-link">SODAR
manual</a>.</p>
</div>
<div class="section level4">
<h4 id="landing-zones">Landing Zones:<a class="anchor" aria-label="anchor" href="#landing-zones"></a>
</h4>
<p>Landing zones in SODAR serve as temporary, user-specific areas within
iRODS for file uploads, providing full write access to the user. Files
are uploaded to these landing zones, after which SODAR performs
validation and transfers them to a permanent, read-only sample data
repository. The Landing Zones application facilitates the management of
these uploads, accessible through the project sidebar or the SODAR home
page. Should validation or upload fail, the process is reversed to allow
corrections, ensuring the integrity of data in the repository.</p>
</div>
<div class="section level4">
<h4 id="sample-sheets">Sample sheets:<a class="anchor" aria-label="anchor" href="#sample-sheets"></a>
</h4>
<p>In SODAR, sample sheets represent the metadata for study design
within research projects. The Sample Sheets application is used for
importing, browsing, and editing this metadata, and it allows users to
access related files stored in the iRODS system. Access to the Sample
Sheets app is obtained through the project sidebar or via an icon on the
SODAR home page.</p>
</div>
<div class="section level4">
<h4 id="time-line">Time line:<a class="anchor" aria-label="anchor" href="#time-line"></a>
</h4>
<p>The Timeline application in SODAR serves as a log that tracks project
activities, such as member role assignments, file transfers from landing
zones, updates to sample sheets or errors. Each recorded event on the
timeline includes a timestamp, the type and origin of the event, the
user who initiated it, and a detailed description with links to related
objects and applications.</p>
</div>
<div class="section level4">
<h4 id="rest-api">REST-API<a class="anchor" aria-label="anchor" href="#rest-api"></a>
</h4>
<p>TODO</p>
</div>
</div>
<div class="section level3">
<h3 id="sodar-data-workflow-step-by-step-usage">SODAR data workflow step-by-step usage<a class="anchor" aria-label="anchor" href="#sodar-data-workflow-step-by-step-usage"></a>
</h3>
<p><img src="../fig/clipboard-3042118772.png" width="847" class="figure"> Figure 4.
SODAR data workflow.</p>
<ol style="list-style-type: decimal">
<li><p>Create a project <a href="https://sodar-test.dieterichlab.org/project/project/create" class="external-link uri">https://sodar-test.dieterichlab.org/project/project/create</a></p></li>
<li><p>Create a subproject <a href="https://sodar-test.dieterichlab.org/project/project/create/f2688cf8-67f2-450d-ae69-97d5fda66faf" class="external-link uri">https://sodar-test.dieterichlab.org/project/project/create/</a>&lt;project_uid&gt;</p></li>
<li>
<p>Create a sample sheet: Sample sheets &gt; Sheet operations &gt;
Create from template</p>
<ol style="list-style-type: decimal">
<li>Select ISA-Tab Template (Bulk RNA sequencing ISA-tab template)</li>
<li>Import Sample Sheets from ISA-Tab</li>
</ol>
</li>
<li><p>Activate the irods server on Sample Sheet section</p></li>
<li>
<p>Upload files</p>
<ol style="list-style-type: decimal">
<li><p>Use &lt;<a href="https://github.com/bihealth/cubi-tk" class="external-link">cubi-tk</a>
<a href="https://github.com/bihealth/cubi-tk/blob/0f7f5b2d46d0f955eb70faf8741cc44033fe63ce/docs_manual/man_sodar_ingest.rst#L4" class="external-link">sodar
ingest</a>&gt; cubi-tk sodar ingest –sodar-url <url> –sodar-api-token
<api_token><file><landing_zone_uid></landing_zone_uid></file></api_token></url></p></li>
<li><p>Use iRODs &lt;<a href="https://docs.irods.org/master/icommands/user/#iput" class="external-link">iput</a>&gt;</p></li>
</ol>
</li>
<li><p>Call SODAR validation</p></li>
</ol>
</div>
</section><section id="faq-and-pratical-notes"><h2 class="section-heading">FAQ and pratical notes:<a class="anchor" aria-label="anchor" href="#faq-and-pratical-notes"></a>
</h2>
<hr class="half-width">
<div class="section level3">
<h3 id="how-to-access-irods">How to access iRODS:<a class="anchor" aria-label="anchor" href="#how-to-access-irods"></a>
</h3>
<p>iRODS directories can be accessed . <a href="https://sodar-server.readthedocs.io/en/latest/data_transfer_irods.html" class="external-link uri">https://sodar-server.readthedocs.io/en/latest/data_transfer_irods.html</a></p>
</div>
<div class="section level3">
<h3 id="why-cubi-tk-sodar-ingest-fasq-is-not-working">Why &lt;cubi-tk sodar ingest-fasq&gt; is not working?<a class="anchor" aria-label="anchor" href="#why-cubi-tk-sodar-ingest-fasq-is-not-working"></a>
</h3>
<p>The cubi-tk is a command line tool that allows to interact with SODAR
server. The command <code>cubi-tk sodar ingest-fasq</code> because of
configuration error. Use &lt;cubi-tk sodar ingest&gt;. See
implementation details on <a href="https://github.com/bihealth/cubi-tk/blob/0f7f5b2d46d0f955eb70faf8741cc44033fe63ce/cubi_tk/sodar/ingest.py" class="external-link">cubi-tk</a>.</p>
<p>? URL for sheet synchronization</p>
<p>? iRODS public access ticket</p>
<p>? IGV session genome</p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points<a class="anchor" aria-label="anchor" href="#keypoints1"></a>
</h3>
<div class="callout-content">
<ul>
<li>TODO</li>
</ul>
</div>
</div>
</div>
<!-- 
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use. 
 -->
</div>
</section></section><section id="aio-data_management_plan"><p>Content from <a href="data_management_plan.html">Data Management Plan</a></p>
<hr>
<p>Last updated on 2024-03-22 |
        
        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/episodes/data_management_plan.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 0 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>None</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Establishing a data management strategy within the CRC.</li>
<li>Detailing roles and responsibilities for data management.</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section id="overview"><h2 class="section-heading">Overview<a class="anchor" aria-label="anchor" href="#overview"></a>
</h2>
<hr class="half-width">
<p>This Data Management Plan outlines the strategies and procedures for
managing high-throughput data within the CRC1550’s integrated
infrastructure. The CRC-INF plan is centered on the consolidation of
interfaces, procedures, and interoperability standards (Aim 1) and the
development of generic bioinformatics workflows for the standard
analysis of multimodal data sets from CRC projects (Aim 2). The primary
objective of the data management team is to ensure the efficient
management of unprocessed and processed data produced by high-throughput
experiments by providing services (and interfaces) to manage and store
data and metadata.</p>
<figure><img src="https://lh7-us.googleusercontent.com/HIZrk0S_WDLJXpKYFcl6BfZY5or_grINIHr6JNW_5mZmpb3xLxy6_aQEcPiEAmGwL_pS6LpaJ9aPpHUEbDZ-DSDUpPtE_cciZNdvwZGSsqYUJ9br8Bzr_bacQ2UdG1-WFKlsMBUu9LBpergxc4EtppU" class="figure mx-auto d-block"></figure><p>The data generated in high-throughput experiments can be categorized
into two main types:</p>
<p><strong>Unprocessed (or raw) data</strong> refers to the unprocessed
data directly obtained from high-throughput experiments. This includes
the output from sequencing machines. The data management team will
establish protocols for the collection, storage, and initial processing
of raw data to ensure its integrity and reliability. There is a general
rule of thumb for </p>
<p><strong>Processed data</strong> represents the results obtained after
applying various bioinformatics and statistical analysis techniques to
the raw data. This includes sequences filtered by quality, identified
genetic variants, and derived annotations, such as genes called
differentially expressed. The data management team will work closely
with the bioinformatics team to establish workflows and standardize
processing methods to ensure consistent and reproducible results.</p>
<div class="section level3">
<h3 id="roles-and-responsibilities">
<strong>Roles and responsibilities</strong><a class="anchor" aria-label="anchor" href="#roles-and-responsibilities"></a>
</h3>
<p>The data owner and data manager must work together to ensure data
flow, documentation, and data management best practices throughout the
project. To keep high-throughput data secure, accessible, and usable,
these roles must communicate regularly.</p>
<p><br><br></p>
<p><strong>Data Owner</strong>:</p>
<p><br></p>
<ul>
<li><p>High-throughput experiment data transfer: The data owner
transfers primary data from high-throughput experiments to the data
management team. This includes organizing and documenting data.</p></li>
<li><p>Access control change communication: The data owner should notify
the infrastructure management team of access control changes. This
ensures project-compliant access permissions.</p></li>
<li><p>In case of record update: The data owner must keep data and
metadata up-to-date. Throughout the project lifecycle, document data and
metadata changes. Data integrity and completeness should be ensured by
the data owner and manager.</p></li>
</ul>
<p><br></p>
<p><strong>Data Manager:</strong></p>
<p><br></p>
<ul>
<li><p>Validates high-throughput data and metadata, this involves
checking the data owner’s metadata for MINSEQE and ISA compliance,
inconsistencies, and missing information.</p></li>
<li><p>Storage and Backup. The data manager stores and backs up raw
data, processed data, and metadata. To preserve data and recover from
disasters, this requires proper storage systems, data organization, and
regular backups.</p></li>
<li><p>Maintain a healthy data management system. This includes system
monitoring, troubleshooting, and updating software and security
patches.</p></li>
<li><p>The data manager transfers data and metadata from the data
management system to bioinformatics teams or other stakeholders. This
may involve exporting data in required formats, ensuring data integrity
during transfer, and working with bioinformatics teams to meet their
needs.</p></li>
</ul>
</div>
</section><section id="managing-metadata"><h2 class="section-heading">Managing metadata <a class="anchor" aria-label="anchor" href="#managing-metadata"></a>
</h2>
<hr class="half-width">
<p>To enable the unambiguous interpretation and facilitate the
reproducibility of experiment results, we will adopt the Minimum
Information about a high-throughput nucleotide SEQuencing Experiment
(<strong>MINSEQE</strong>) guidelines. These guidelines provide a
framework for capturing essential metadata associated with
high-throughput experiments and are the recommended community standard.
Similar to the MIAME guidelines for microarray experiments, MINSEQE will
be followed to ensure comprehensive metadata collection and consists of
the following items:</p>
<ul>
<li><p>metadata</p></li>
<li><p>sequence read data</p></li>
<li><p>final processed data </p></li>
<li><p>experimental protocols</p></li>
</ul>
<p><br></p>
<p><strong>ISA framework</strong>: this framework aims to simplify </p>
<ul>
<li><p><strong>I</strong>nvestigation: the project context</p></li>
<li><p><strong>S</strong>tudy: a unit of research</p></li>
<li><p><strong>A</strong>ssay: analytical measurement</p></li>
</ul>
<p><br></p>
<p><strong>Data Portal</strong></p>
<p>Task: provide granularity access control to certain data. Initially,
data is available only to data owners. After authorization, data should
be made internally and openly accessible. </p>
<p><br></p>
<p><strong>Making metadata interoperable</strong></p>
<p>Task: To enhance the interoperability of metadata, the data
management team will utilize standardized metadata formats, such as XML
or JSON, and adopt controlled vocabularies and ontologies where
applicable. This will enable efficient data discovery, integration, and
sharing across different projects and platforms.</p>
<p><br><br></p>
<p><strong>Data security and backup</strong></p>
<p>Task<strong>:</strong> we will establish protocols for data security
and backup to ensure that data is protected against loss or corruption.
The protocols will include measures for data backup.</p>
<p><br></p>
<p><strong>Ethical aspect</strong></p>
<p>Currently, we only store data obtained from animal samples for
genetic and disease modeling. Therefore, there are no ethical or legal
issues that can affect data sharing, assuming a vote on animal ethics
has been obtained by the data provider.</p>
<p><br></p>
<p><strong>Costs</strong></p>
<p>The data management team will allocate appropriate resources,
including storage infrastructure, computational power, and personnel, to
support the storage and metadata processing. Regular assessments will be
conducted to ensure the scalability and sustainability of the
infrastructure and its services.</p>
<p><br></p>
<p><strong>OpenBis</strong></p>
<p>To fulfill these tasks, we are using the Biological Information
System (OpenBIS), a flexible and extensible data management system that
can be utilized to accomplish several aspects of your Data Management
Plan. Here’s how OpenBIS can be used to support the different
sections:</p>
<p>OpenBIS includes a web-based interface that can serve as a data
portal. It allows authorized users to search, browse, and access data
and metadata stored in the system. As a metadata management system, it
provided a structured metadata model. We are working to define metadata
templates based on the MINSEQE guidelines and the ISA framework within
OpenBIS. This ensures that the required metadata fields are captured for
each experiment, study, and assay. In addition, metadata quality
checklist should be implemented </p>
<p>The data portal can be customized to provide specific functionalities
for data visualization and analysis, enabling users to perform basic
data exploration tasks.</p>
<p>The portal supports access control mechanisms, allowing us to define
user roles and permissions. We can configure access restrictions to
ensure that consortium members have the appropriate level of access to
data and metadata within the system. OpenBIS enables collaboration by
providing controlled sharing and collaboration features among authorized
users. In addition, OpenBIS can integrate with public repositories that
work as  external data sharing platforms, such as Zenodo, that generate
persistent identifiers (e.g., DOI) for datasets, facilitating easy
referencing and citation.</p>
<p>OpenBIS includes built-in security features to protect data integrity
and confidentiality. It supports user authentication and authorization,
data encryption, and audit trails to track system activities.
Additionally, OpenBIS can be integrated with backup systems to ensure
regular data backups and disaster recovery.</p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points<a class="anchor" aria-label="anchor" href="#keypoints1"></a>
</h3>
<div class="callout-content">
<ul>
<li>TODO</li>
</ul>
</div>
</div>
</div>
<!-- 
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use. 
 -->
</section></section><section id="aio-sra"><p>Content from <a href="sra.html">SRA</a></p>
<hr>
<p>Last updated on 2024-03-22 |
        
        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/episodes/sra.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 12 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>What is SRA and how to use it for data deposition?</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Get to know the SRA database</li>
<li>Understand the requiriments for data deposition</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section id="introduction"><h2 class="section-heading">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h2>
<hr class="half-width">
<p>The Sequence Read Archive is a public repository that contains
high-throughput sequencing data. It is a part of the International
Nucleotide Sequence Database Collaboration (INSDC), which includes
several institutes. Data from all life forms is included in the
repository, as well as metagenomics and ecological studies. SRA stores
raw sequencing data and alignment. We use the SRA to deposit, which is
usually mandatory as part of a paper publication.</p>
<p>It aims to establish a central repository for next-generation
sequencing data, linking to resources that reference or utilize this
data. The repository allows in tracking project metadata for studies and
experiments. There’s a focus on facilitating flexible submission and
retrieval of ancillary data providing normalized data structures.
Additionally, the objectives include decoupling the submission process
from content and laying the groundwork for interactive user submissions
and data retrieval.</p>
</section><section id="sra-concepts"><h2 class="section-heading">SRA concepts:<a class="anchor" aria-label="anchor" href="#sra-concepts"></a>
</h2>
<hr class="half-width">
<p>SRA metadata concept separates experimental data from its metadata,
organizing the latter into a structured hierarchy:</p>
<p>— Study: Represents a collection of experiments aimed towards a
common goal, serving as an overarching project that encapsulates the
purpose and scope of the research conducted. — Experiment: Refers to a
specific set of laboratory procedures applied to input material,
designed to achieve an anticipated result. Each experiment is a
component of a study, contributing to its overall objective. Can also be
interpreted as a series of experimental protocols. — Sample: The focus
of an experiment, which can be a single sample or multiple samples
grouped together. The results of an experiment are articulated based on
these samples, detailing the outcomes for individual samples or their
collective as defined by the experimental setup. Sample are not equal a
individuals, as one individual can provide multiple samples (i,e, right
and left ventricle). — Run: Denotes the actual outcomes or results of
the experiment. These runs encompass the data collected from a sample or
a group of samples linked to a specific experiment. Or simply the the
output of the research process. — Submission: A submission encompasses a
bundle of metadata and data objects, coupled with instructions on how
the submission should be handled. This package facilitates the organized
and controlled entry of both experimental data and its corresponding
metadata into the repository, ensuring that the data can be accurately
classified, accessed, and utilized.</p>
<p>Understanding these concepts and the structure of the organization is
important for providing detailed yet direct metadata on data on the
various steps involved in generating a data set.</p>
</section><section id="accessing-the-sra"><h2 class="section-heading">Accessing the SRA<a class="anchor" aria-label="anchor" href="#accessing-the-sra"></a>
</h2>
<hr class="half-width">
<p>The SRA database can be accessed through the <a href="https://www.ncbi.nlm.nih.gov/sra" class="external-link">NCBI website</a>.</p>
<p>Although discussing data access is outside the scope of this
material, it is important to note that data access is a key aspect of
the FAIR principles. This means that with the right metadata, data can
be reused, generating more follow-up studies and citations.</p>
<p>Significant to note that not all data on the SRA is public, some data
is private and only accessible to the submitter. Data can also be
embargoed for a certain period of time. Recently, SRA started to use
placeholder as a way to keep track of data produced by large cohort and
require controlled access.</p>
</section><section id="data-deposition"><h2 class="section-heading">Data deposition<a class="anchor" aria-label="anchor" href="#data-deposition"></a>
</h2>
<hr class="half-width">
<p>There multiple ways to deposit data on the SRA, the most common is
through the submission portal. This approach is detailed <a href="https://www.ncbi.nlm.nih.gov/sra/docs/submitportal/" class="external-link">here</a>. At
the CRC, we plan to streamline this process using a programmatic
approach for data submission, as detailed <a href="https://cloud.crc1550.com/index.php/s/NYkwKpMbHP3GTBn" class="external-link">here</a>.</p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points<a class="anchor" aria-label="anchor" href="#keypoints1"></a>
</h3>
<div class="callout-content">
<ul>
<li><p>Comprehensive Repository: The SRA serves as a comprehensive
public repository for high-throughput sequencing data across all life
forms, supporting a wide range of genomic research by providing access
to raw sequencing data and alignments.</p></li>
<li><p>Structured Metadata Organization: The organization of SRA
metadata into a structured hierarchy—comprising Studies, Experiments,
Samples, Runs, and Submissions—facilitates precise and efficient data
deposition, retrieval, and analysis, underscoring the importance of
detailed metadata in the scientific research process.</p></li>
<li><p>Access and Deposition Flexibility: While access to the SRA
database is facilitated through the NCBI website, the database also
highlights the principles of controlled access and privacy for certain
data sets, alongside offering multiple pathways for data deposition to
accommodate the needs of diverse research projects and goals.</p></li>
</ul>
</div>
</div>
</div>
<!-- 
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use. 
 -->
</section></section>
</div>
    </main>
</div>
<!-- END  : inst/pkgdown/templates/content-extra.html -->

      </div>
<!--/div.row-->
      		<footer class="row footer mx-md-3"><hr>
<div class="col-md-6">
        <p>This lesson is subject to the <a href="CODE_OF_CONDUCT.html">Code of Conduct</a></p>
        <p>
        
        <a href="https://github.com/tbrittoborges/omics_metadata/edit/main/README.md" class="external-link">Edit on GitHub</a>
        
	
        | <a href="https://github.com/tbrittoborges/omics_metadata/blob/main/CONTRIBUTING.md" class="external-link">Contributing</a>
        | <a href="https://github.com/tbrittoborges/omics_metadata/" class="external-link">Source</a></p>
				<p><a href="https://github.com/tbrittoborges/omics_metadata/blob/main/CITATION" class="external-link">Cite</a> | <a href="mailto:thiago.brittoborges@uni-heidelberg.de">Contact</a> | <a href="https://carpentries.org/about/" class="external-link">About</a></p>
			</div>
			<div class="col-md-6">
        
        <p>Materials licensed under <a href="LICENSE.html">CC-BY 4.0</a> by the authors</p>
        
        <p>Template licensed under <a href="https://creativecommons.org/licenses/by-sa/4.0/" class="external-link">CC-BY 4.0</a> by <a href="https://carpentries.org/" class="external-link">The Carpentries</a></p>
        <p>Built with <a href="https://github.com/carpentries/sandpaper/tree/0.16.3" class="external-link">sandpaper (0.16.3)</a>, <a href="https://github.com/carpentries/pegboard/tree/0.7.4" class="external-link">pegboard (0.7.4)</a>, and <a href="https://github.com/carpentries/varnish/tree/1.0.1" class="external-link">varnish (1.0.1)</a></p>
			</div>
		</footer>
</div> <!-- / div.container -->
	<div id="to-top">
		<a href="#top">
      <i class="search-icon" data-feather="arrow-up" role="img" aria-label="Back To Top"></i><br><!-- <span class="d-none d-sm-none d-md-none d-lg-none d-xl-block">Back</span> To Top --><span class="d-none d-sm-none d-md-none d-lg-none d-xl-block">Back</span> To Top
		</a>
	</div>
  <script type="application/ld+json">
    {
  "@context": "https://schema.org",
  "@type": "TrainingMaterial",
  "@id": "https://tbrittoborges.github.io/omics_metadata/instructor/aio.html",
  "inLanguage": "en",
  "dct:conformsTo": "https://bioschemas.org/profiles/TrainingMaterial/1.0-RELEASE",
  "description": "A Carpentries Lesson teaching foundational data and coding skills to researchers worldwide",
  "keywords": "data, metadata, bioinformatics, data management",
  "name": "All in One View",
  "creativeWorkStatus": "active",
  "url": "https://tbrittoborges.github.io/omics_metadata/instructor/aio.html",
  "identifier": "https://tbrittoborges.github.io/omics_metadata/instructor/aio.html",
  "dateCreated": "2024-02-14",
  "dateModified": "2024-03-22",
  "datePublished": "2024-03-22"
}

  </script><script>
		feather.replace();
	</script><!-- Matomo
    2022-11-07: we have gotten a notification that we have an overage for our
    tracking and I'm pretty sure this has to do with Workbench usage.
    Considering that I am not _currently_ using this tracking because I do not
    yet know how to access the data, I am turning this off for now.
  <script>
    var _paq = window._paq = window._paq || [];
    /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
    _paq.push(["setDocumentTitle", document.domain + "/" + document.title]);
    _paq.push(["setDomains", ["*.preview.carpentries.org","*.datacarpentry.github.io","*.datacarpentry.org","*.librarycarpentry.github.io","*.librarycarpentry.org","*.swcarpentry.github.io", "*.carpentries.github.io"]]);
    _paq.push(["setDoNotTrack", true]);
    _paq.push(["disableCookies"]);
    _paq.push(['trackPageView']);
    _paq.push(['enableLinkTracking']);
    (function() {
          var u="https://carpentries.matomo.cloud/";
          _paq.push(['setTrackerUrl', u+'matomo.php']);
          _paq.push(['setSiteId', '1']);
          var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
          g.async=true; g.src='https://cdn.matomo.cloud/carpentries.matomo.cloud/matomo.js'; s.parentNode.insertBefore(g,s);
        })();
  </script>
  End Matomo Code -->
</body>
</html><!-- END:   inst/pkgdown/templates/layout.html-->

